<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GN_MR_IPZC3H4</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/FabioBedin/GN_MR_IPZC3H4">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-09-03
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GN_MR_IPZC3H4/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210721code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210721)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210721code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210721)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomFabioBedinGNMRIPZC3H4tree105f46b499ceda0505d5a627dfdb14718c178c67targetblank105f46ba"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/FabioBedin/GN_MR_IPZC3H4/tree/105f46b499ceda0505d5a627dfdb14718c178c67" target="_blank">105f46b</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomFabioBedinGNMRIPZC3H4tree105f46b499ceda0505d5a627dfdb14718c178c67targetblank105f46ba" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/FabioBedin/GN_MR_IPZC3H4/tree/105f46b499ceda0505d5a627dfdb14718c178c67" target="_blank">105f46b</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  code/custom_functions.R
    Untracked:  code/custom_functions2.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/expdesign_all.tsv
    Untracked:  data/expdesign_old.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  data/proteinGroups_INPUT.txt
    Untracked:  data/proteinGroups_all.txt
    Untracked:  data/proteinGroups_old.txt
    Untracked:  output/Significant.txt
    Untracked:  output/Significant_old.txt
    Untracked:  output/enrichmentmap.txt
    Untracked:  output/network_FC_pvalue.png
    Untracked:  output/network_enrichmentmap.png
    Untracked:  output/new_proteins_identify.txt
    Untracked:  output/old_proteins_identify.txt
    Untracked:  output/results_GN_MR_IPZC3H4_210721.xlsx
    Untracked:  output/results_GN_MR_IPZC3H4_all_210721.xlsx
    Untracked:  output/results_GN_MR_IPZC3H4_old_210721.xlsx
    Untracked:  output/results_table.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Template_LFQ.Rmd</code>) and HTML (<code>docs/Template_LFQ.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/FabioBedin/GN_MR_IPZC3H4/blob/105f46b499ceda0505d5a627dfdb14718c178c67/analysis/Template_LFQ.Rmd" target="_blank">105f46b</a>
</td>
<td>
FabioBedin
</td>
<td>
2021-09-03
</td>
<td>
add the new Template_LFQ file
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions2.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)/2)
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<div id="ipzc3h4-vs-ipev" class="section level1">
<h1><strong>IPZC3H4</strong> vs <strong>IPEV</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the <strong>proteinGroups</strong> obtained by <a href="https://www.maxquant.org/">MaxQuant</a> software, I remove proteins that are marked form the software as potential contaminant, only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins identified by less than 2 peptides. (default behavior)<br />
Now, we generate some quality control plots to investigate the structure of the dataset and observe the effect of filters. (aggiungere il tipo di filtro che faccio)</p>
<div id="quality-control-plots" class="section level2 tabset tabset-fade">
<h2><strong>1. Quality control plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3>Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples <em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3>Distributions</h3>
<p>This plot is useful for checking the distributions of the samples <em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3>Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and before User define</em> filters. Notice that the percentages of missingness are provided in the data. These are accurate to 1 decimal place. Also the dataset is arranged by columns with most missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3>Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the dispersion of data points in a data series around the mean. The coefficient of variation represents the ratio of the standard deviation to the mean, and it is a useful statistic for comparing the degree of variation from one data series to another, even if the means are drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within the 2 conditions or in common between all condition.</p>
<div id="ipzc3h4" class="section level3">
<h3>IPZC3H4</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="ipev" class="section level3">
<h3>IPEV</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry proteomics experiment would provide absolute abundance measurements for a whole proteome, across varying conditions. Unfortunately, this ideal is not realized. Measurements are made on peptides requiring an inferential step to obtain protein level estimates. The inference is complicated by experimental factors that necessitate relative abundance estimation and result in widespread non-ignorable missing data. Relative abundance on the log scale takes the form of parameter contrasts. In a complete-case analysis, contrast estimates may be biased by missing data and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- mix_imputation_mean(data_filt, cond1 = cond_1, cond2 = cond_2)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics are used for the differential enrichment analysis (or differential expression analysis) and False discovery rate (FDR) is ajusted with <em>Benjamini-Hochberg</em> procedure. Significant proteins are are define with <strong>FDR = 0.05</strong> and <strong>Fold change = 1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>223</strong> significant proteins in <strong>IPZC3H4</strong> vs <strong>IPEV</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a number of functions. These visualizations assist in the determination of the optimal cutoffs to be used, highlight the most interesting samples and contrasts, and pinpoint differentially enriched/expressed proteins.</p>
<div id="visualization-of-the-results" class="section level3 tabset tabset-fade">
<h3><strong>4.1. Visualization of the results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data. This can be very useful to observe batch effects, such as clear differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between the two samples (x axis) and their corresponding adjusted p value (y axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-and-unique" class="section level4">
<h4>Volcano and unique</h4>
<pre class="r"><code>plot_volcano_2(dep, contrast=test, add_names=T, label_size=5, adjusted = F, unique_vec = uni_name)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/volcano-unique-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4">
<h4>Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant proteins (rows) in all samples (columns). This allows to see general trends, for example if one sample or replicate is really different compared to the others. Additionally, the clustering of samples (columns) can indicate closer related samples and clustering of proteins (rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by p.value. Proteins that are UP regulated are colored with a red box in Fold change column, in the orhter hand proteins that are DOWN regulated are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-2a0dd1ba32e0541469b5" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2a0dd1ba32e0541469b5">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223"],["CFL1","WDR82","HDAC1","SRSF4","ZC3H4","CDKN2AIP","MSH6","SEC24C","SFPQ","TLE3","TNKS1BP1","ABCF3","AFTPH","CAPN7","DYNC1LI1","NUP214","RABGGTA","RPL32","TDRD9","NKRF","KDM1A","GIGYF2","MTA2","RPL23A","RPS24","SLC16A3","CBS","RPS15","ANXA10","HNRNPR","ILVBL","UBR4","FERMT2","RANBP2","HK1","PANK4","NUP133","USP10","G3BP1","NUP88","DCTN1","SMG7","UNC45A","ARCN1","PKP2","CARM1","FNDC3B","SDF4","DSG2","TTC27","CSNK1A1","ITGB4","LUZP1","MCCC1","MYCBP2","PDE3A","NEFH","MRPL44","PPP1R13L","PIK3C2A","CPEB4","RPL36AL","ZCCHC8","EIF2B5","FARP2","IQSEC1","SKIV2L","TBL1XR1","TCF12","XPOT","DCAF7","RRBP1","KRT18","WDR62","NUP205","RBM14","DNAJC13","THOC6","FAM83H","FARP1","HSPA2","ZNF281","UBAP2L","VIM","TIMM13","NES","CDC45","MCM3AP","TUBB4A","CAP2","RCN3","EIF2S1","IMPA2","POLR1C","PHLDB2","PSMA4","PSMF1","PLEC.1","NUP62","ITGA6","ARHGEF2","FAM208B","AHR","CTSD","IDH1","KPNA2","SNRPD2","P4HA1","FAM83B","RPTOR","VPS51","PRKDC","DPM1","HDAC2","AHNAK","BAG3","AHSA1","DYNLL1","INPP5K","MTUS1","PRKAR1A","DYNC1H1","TRIM27","SEC23A","PARP2","CTNND1","DCD","PDIA3","CYP1B1","ABCF2","POLD1","SEC16A","IMPDH2","PKP3","DDX50","HEATR1","MAP7","RCN1","ESYT2","CAD","DDX5","KCTD17","MYO1E","THOP1","PRSS3","PSMD4.1","DSG1","EIF4G2","CPS1","SKIV2L2","EIF3L","FLNC","EIF3D","RICTOR","RBMXL1","PNPLA6","WTAP","TIMM50","CALU","INA","NUP188","RPL31","ACTB","S100A8","CASP14","EIF4A3","TKT","DSP","MON2","ALPP","TMPO","BAG2","RBMX","REST","P4HB","U2AF1","TMPO.1","TFRC","NUP93","AKAP8L","CLINT1","XRCC6","CNP","EMD","LMO7","TCP1","BNC2","EPPK1","FARSA","MCM7","SERPINB12","ZC3H18","FMR1","CHD9","KIF11","SRSF7","TRMT1","HNRNPM","TRIM21","RAN","RCN2","YTHDF2","SPIN3","AZGP1","XRCC5","SYNRG","RIF1","TRIP13","CAPN1","PAICS","PKM","RPS16","TGM3","ACAT1","PC","MYO1C","MSH2","GAPDH","GNB2","NO66","PRPF31","RPS27","GANAB"],["1.6e-08","1.3e-08","1.4e-07","1.1e-07","1.2e-07","4.7e-07","4.0e-07","4.0e-07","4.3e-07","4.5e-07","4.3e-07","8.3e-07","8.1e-07","8.8e-07","7.5e-07","6.2e-07","8.9e-07","7.9e-07","8.6e-07","1.2e-06","1.3e-06","1.8e-06","1.8e-06","1.7e-06","1.8e-06","1.7e-06","2.0e-06","2.0e-06","2.3e-06","2.3e-06","2.3e-06","2.3e-06","2.6e-06","2.5e-06","2.7e-06","2.9e-06","3.7e-06","3.8e-06","3.9e-06","4.4e-06","4.7e-06","4.6e-06","5.9e-06","6.2e-06","6.5e-06","6.8e-06","8.3e-06","8.2e-06","9.6e-06","1.1e-05","1.2e-05","1.2e-05","1.2e-05","1.3e-05","1.3e-05","1.3e-05","1.6e-05","1.7e-05","1.9e-05","1.9e-05","2.0e-05","2.3e-05","2.4e-05","2.6e-05","2.8e-05","2.8e-05","2.7e-05","2.7e-05","2.6e-05","2.9e-05","3.5e-05","3.8e-05","4.1e-05","4.1e-05","4.8e-05","4.8e-05","5.2e-05","5.2e-05","5.5e-05","5.8e-05","6.0e-05","6.0e-05","7.5e-05","7.8e-05","1.0e-04","1.1e-04","1.1e-04","1.3e-04","1.3e-04","1.4e-04","1.6e-04","1.7e-04","1.7e-04","2.0e-04","2.2e-04","2.2e-04","2.2e-04","2.3e-04","3.1e-04","3.3e-04","3.6e-04","3.9e-04","4.1e-04","4.1e-04","4.6e-04","4.5e-04","4.9e-04","5.4e-04","5.8e-04","6.1e-04","7.0e-04","7.1e-04","8.2e-04","8.9e-04","9.1e-04","9.4e-04","9.5e-04","9.8e-04","9.8e-04","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.6e-03","1.6e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.5e-03","2.6e-03","2.7e-03","2.7e-03","2.7e-03","2.8e-03","2.9e-03","3.1e-03","3.3e-03","3.3e-03","3.5e-03","3.6e-03","3.7e-03","3.6e-03","3.7e-03","3.7e-03","3.9e-03","3.9e-03","4.0e-03","4.1e-03","4.1e-03","4.3e-03","4.3e-03","4.4e-03","4.5e-03","4.5e-03","5.0e-03","4.9e-03","4.9e-03","5.1e-03","5.4e-03","5.5e-03","5.5e-03","5.6e-03","5.8e-03","5.8e-03","6.0e-03","6.1e-03","6.2e-03","6.3e-03","6.4e-03","6.6e-03","6.9e-03","6.9e-03","6.9e-03","7.1e-03","7.1e-03","7.1e-03","7.5e-03","7.7e-03","7.7e-03","7.7e-03","7.8e-03","7.9e-03","8.2e-03","8.4e-03","8.4e-03","8.4e-03","8.4e-03","8.6e-03","8.6e-03","9.2e-03","9.3e-03","9.3e-03","9.9e-03","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.4e-02","1.5e-02","1.4e-02","1.5e-02"],["6.1e-06","6.1e-06","2.1e-05","2.1e-05","2.1e-05","3.2e-05","3.2e-05","3.2e-05","3.2e-05","3.2e-05","3.2e-05","3.4e-05","3.4e-05","3.4e-05","3.4e-05","3.4e-05","3.4e-05","3.4e-05","3.4e-05","4.3e-05","4.7e-05","5.2e-05","5.2e-05","5.2e-05","5.2e-05","5.2e-05","5.3e-05","5.3e-05","5.4e-05","5.4e-05","5.4e-05","5.4e-05","5.6e-05","5.6e-05","5.8e-05","5.9e-05","7.3e-05","7.3e-05","7.4e-05","8.1e-05","8.2e-05","8.2e-05","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.3e-04","1.3e-04","1.5e-04","1.6e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","2.1e-04","2.2e-04","2.4e-04","2.4e-04","2.4e-04","2.8e-04","2.8e-04","3.0e-04","3.0e-04","3.0e-04","3.0e-04","3.0e-04","3.0e-04","3.0e-04","3.7e-04","3.9e-04","4.1e-04","4.1e-04","4.7e-04","4.7e-04","5.0e-04","5.0e-04","5.2e-04","5.3e-04","5.4e-04","5.4e-04","6.7e-04","6.9e-04","8.9e-04","9.3e-04","9.7e-04","1.1e-03","1.1e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","2.3e-03","2.4e-03","2.7e-03","2.8e-03","2.9e-03","2.9e-03","3.2e-03","3.2e-03","3.4e-03","3.7e-03","3.9e-03","4.1e-03","4.7e-03","4.7e-03","5.4e-03","5.8e-03","5.8e-03","6.0e-03","6.0e-03","6.1e-03","6.1e-03","7.7e-03","8.0e-03","8.0e-03","8.0e-03","8.1e-03","9.4e-03","9.5e-03","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.2e-02","2.2e-02","2.2e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.7e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.4e-02","3.4e-02","3.4e-02","3.6e-02","3.6e-02","3.6e-02","3.8e-02","3.8e-02","3.8e-02","3.9e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.1e-02","4.2e-02","4.2e-02","4.2e-02","4.5e-02","4.8e-02","4.8e-02","4.8e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[6.9,10.3,5.81,5.22,9.15,5.34,4.68,5.04,4.76,5.91,5.05,4.49,4.95,5.4,5.19,6.97,5.92,5.44,4.18,3.68,4.05,3.89,4.44,4.42,4.58,4.69,3.62,4.12,3.55,3.47,3.49,3.5,4.03,3.38,4.15,5.04,5.43,3.76,3.22,6.18,3.02,4.42,3.16,4.35,7.35,4.21,3.99,5.76,5.84,4.72,5.59,6.44,5.73,4.69,2.83,7.95,6,2.46,6.14,2.91,4.7,5.32,4.82,2.85,4.2,4.36,4.46,4.51,5.62,4.52,4.33,3.5,4.57,3.24,3.37,5.55,3.3,2.48,7.57,2.69,3.63,5.18,7.17,3.84,5.34,7.28,4.74,5.39,2.89,-2.74,7.23,2.71,3.29,2.16,2.27,-1.77,3.78,4.67,5.18,4.08,6.16,-2.39,3.61,-2.3,-4.39,2.47,1.78,3.81,5.62,4.12,2.14,3.1,2.12,2.52,4.94,3.55,3.66,4.32,4.22,4.57,-1.51,2.41,3.45,2.01,-2.02,2.84,-1.9,-1.93,1.53,2.41,2.34,4.04,-2.22,5.48,-1.56,2.43,1.91,3.57,2.37,2.09,1.27,-1.41,2.74,1.5,-1.83,-1.35,-1.89,4.12,1.28,1.88,2.2,2.41,1.08,4.21,-2.14,2.49,-1.17,2.78,6.16,4.5,1.33,1.61,1.45,-1.19,-1.83,-1.11,-1.19,2.4,1.67,3.7,-2.09,1.71,-2.53,-2.02,1.35,-1.6,2.87,1.74,3.34,2.55,4.15,-1.44,-1.5,2,1.87,2.48,-1.69,4.12,2.25,1.64,-1.81,1.76,-1.13,4.47,-1.3,1.01,1.03,1.61,-1.12,1.03,3.13,3.01,-1.52,-1.31,-1.24,2.26,-1.01,2.19,1.94,-1.07,1.83,2.79,-2.21,-1.13,2.56,1.04,1.73,-1.16,1.72,1.91,1.81,2.12,1.56]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF" class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a href="GO:CC" class="uri">GO:CC</a>) and a KEGG ontology.<br />
The enrichment results are visualized with an interactive <strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="ipzc3h4-1" class="section level3 tabset tabset-fade">
<h3><strong>5.1. IPZC3H4</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in IPZC3H4 were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-ed4906484ed42468d86d" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-ed4906484ed42468d86d">{"x":{"data":[{"x":[97.2021122614,150.400043556448,100.323827587156,151.250710982717,100.335534019628,136.317205293129,84.5201437505141,151.270221703503,84.5357523271429,84.5240458946713,151.254613126874,90.4396964369799,131.646338736966,135.88406729168,159.753483101246,134.803173360137,134.79927121598,136.539627510089,135.872360859209,156.592746333918,84.5123394621997,114.153026480258,131.349775781019,166.258357411291,126.487704161153,125.917991114203,175.486928343059,126.483802016996,86.3034236303525,136.321107437286,86.5648672888846,107.7496079183,152.440864950662,90.2445892291201,96.964081467811,137.733683622191,100.24968684817,124.552240659184,84.5045351738853,93.7994425563254,111.140571190903,83.4314455306565,97.2255251263431,88.9412730806168,86.6155951629282,126.639887783284,154.750934291721,83.4431519631281,85.7844384574455,84.5084373180425,96.04707759087,152.148204138872,97.3152744419586,128.8719142412,89.4485518210522,78.4718203068609,104.862021241975,83.681182756717,131.700968755167,144.460980149196,89.7412126328419,89.9090048316013,86.2097721705798,152.249659886959,114.176439345201,89.9324176965445,84.5474587596145,149.568886850966,137.764900775449,93.7213996731814,98.0722904084546,84.5435566154573,83.4977819813289,111.90929358987,85.2303339871237,144.445371572568,86.572671577199,141.440720571527,86.732659487644,104.776174070517,144.453175860882,122.495810688342,89.7295062003703,136.325009581444,123.451836006855,95.4227345257187,114.285699381602,112.369746600419,152.1364977064,81.2657555234129,172.025726475626,113.645747739822,117.348882545001,123.791322548531,162.399136839825,75.4125392876195,131.11174498743,90.7635744020271,124.856607903446,114.246677940031,97.2528401354435,84.8947495896049,163.062501346548,86.8536259565171,177.094611735824,101.252537896569,114.090592173743,75.5803314863789,178.93642377802,107.726195053357,168.162603760003,91.6142418282958,83.6031398735731,123.440129574383,98.1464311474413,171.257004076659,126.507214881939,93.7096932407099,76.4153903360187,160.565129085943,157.322447291313,112.884829629169,140.828083938847,136.360128878858,91.6064375399814,123.787420404374,130.545934084637,93.3077723925187,83.5992377294159,84.4850244530994,98.0332689668826,104.885434106918,86.5687694330418,90.2172742200198,154.828977174865,85.2615511403813,124.801977885245,126.511117026096,157.232697975698,124.103494081107,102.181248205982,93.3116745366759,128.883620673671,110.430380954293,85.2576489962241,161.969900982533,97.8849874889092,83.4938798371717,86.7404637759584,150.150306330388,133.207196399844,154.610457102062,98.9502728438236,154.146101947356,120.704726520189,95.4266366698759,126.370639836437,100.515032650859,75.2252363680741,97.8732810564376,89.7997447951998,84.7425659674743,160.135893228651,136.531823221775,109.298759148706,154.770445012507],"y":[12.3105752986353,12.065838665168,11.7658034854106,10.8643947390237,10.3116027749675,9.81561731376608,9.68702118204921,8.96500258842928,8.96500258842928,8.86127196560435,8.86127196560435,8.72970859670482,8.72104476869807,8.55140493895948,8.42234505933512,8.3865288871022,8.3865288871022,8.27162864676585,8.21646089921152,8.07925453794656,8.04169472602156,7.99758373191002,7.55121788000483,7.49842339467113,7.46855088652263,7.41216453116745,7.41216453116745,7.38641471569454,7.34884219790465,7.28261610202332,7.13786337861458,7.05213505255517,6.73827579266859,6.45100041188767,6.33967247928613,6.14030852458154,5.78407333636006,5.67027254753244,5.50840004211591,5.29789609398834,5.23977641604099,5.21974730996882,5.16099270625131,5.11828076551015,5.05465622594985,5.04926423710088,4.97309880744463,4.94619575563172,4.90982030853916,4.86367896719902,4.79050001053155,4.77790639388096,4.77121997602129,4.63070614686409,4.54465997003449,4.54011120469624,4.45731158947592,4.30855538707161,4.24732330189273,4.21708073957438,4.21708073957438,4.21708073957438,4.2051290337318,4.19617319469723,4.19523060337012,4.12780278023535,4.12577122094037,4.1044749393537,4.0830905448734,4.04386480082389,3.98559123637311,3.9477037045688,3.85553545605873,3.80848765118456,3.76410174009855,3.73642485514894,3.7182480912541,3.6726569946338,3.64943332607394,3.64929271297729,3.64121964235366,3.63388045941937,3.60770760067884,3.49123961449486,3.42056315170398,3.37591981736843,3.37454497414533,3.32877107108601,3.24959199594161,3.24656754469225,3.07165520074946,3.05562893652986,2.93896779861367,2.93761824307814,2.90402012284108,2.90352645091724,2.84887629677326,2.80446936556239,2.78108613884595,2.71987948895642,2.69058319450763,2.67276209906588,2.66034621197629,2.66034621197629,2.65013338989457,2.63719446367458,2.56622421254208,2.53433326701455,2.53246513587474,2.5107942449188,2.50516636387775,2.46142504123961,2.45132021113146,2.424919099629,2.35352612624071,2.34048301879211,2.29259951659038,2.2267040242473,2.20125922816863,2.05407947202757,2.04836825198977,2.0475305569754,2.04094232055316,2.03510387577358,2.01073699174178,1.99767141609882,1.96870763771949,1.95223693861123,1.93727973914825,1.92877604123469,1.91170559185624,1.89808840311663,1.87802318982978,1.86626341010486,1.85997123864166,1.82883014075906,1.81305605497651,1.80725641657665,1.80725641657665,1.77640305978506,1.74937779423234,1.73125055789686,1.72171216632068,1.7027585255768,1.65463425846307,1.62669039621424,1.56076050850407,1.5319412481629,1.53080881346605,1.51878358828862,1.51789843317388,1.50649729404651,1.45058669966329,1.44459151838042,1.42805600682481,1.41623331316765,1.41393283710168,1.38591491132008,1.3773950966273,1.33868124763788,1.33072638893891,1.32724037341065,1.3173887977724,1.31626556150509,1.31487959975785,1.31396209106959],"text":["GO:0016032 (1011) <br> viral process <br> 4.891e-13","GO:0071166 (131) <br> ribonucleoprotein complex localization <br> 8.593e-13","GO:0019080 (237) <br> viral gene expression <br> 1.715e-12","GO:0071426 (130) <br> ribonucleoprotein complex export from nucleus <br> 1.366e-11","GO:0019083 (181) <br> viral transcription <br> 4.880e-11","GO:0051168 (202) <br> nuclear export <br> 1.529e-10","GO:0006405 (129) <br> RNA export from nucleus <br> 2.056e-10","GO:0071431 (34) <br> tRNA-containing ribonucleoprotein complex export from nucleus <br> 1.084e-09","GO:0006409 (34) <br> tRNA export from nucleus <br> 1.084e-09","GO:0006406 (112) <br> mRNA export from nucleus <br> 1.376e-09","GO:0071427 (112) <br> mRNA-containing ribonucleoprotein complex export from nucleus <br> 1.376e-09","GO:0009408 (159) <br> response to heat <br> 1.863e-09","GO:0046907 (1723) <br> intracellular transport <br> 1.901e-09","GO:0051031 (37) <br> tRNA transport <br> 2.809e-09","GO:0097064 (38) <br> ncRNA export from nucleus <br> 3.781e-09","GO:0050658 (183) <br> RNA transport <br> 4.106e-09","GO:0050657 (183) <br> nucleic acid transport <br> 4.106e-09","GO:0051236 (186) <br> establishment of RNA localization <br> 5.350e-09","GO:0051028 (151) <br> mRNA transport <br> 6.075e-09","GO:0075733 (57) <br> intracellular transport of virus <br> 8.332e-09","GO:0006403 (222) <br> RNA localization <br> 9.085e-09","GO:0034605 (119) <br> cellular response to heat <br> 1.006e-08","GO:0046794 (64) <br> transport of virus <br> 2.810e-08","GO:1900034 (79) <br> regulation of cellular response to heat <br> 3.174e-08","GO:0045104 (50) <br> intermediate filament cytoskeleton organization <br> 3.400e-08","GO:0044766 (66) <br> multi-organism transport <br> 3.871e-08","GO:1902579 (66) <br> multi-organism localization <br> 3.871e-08","GO:0045103 (51) <br> intermediate filament-based process <br> 4.108e-08","GO:0006913 (353) <br> nucleocytoplasmic transport <br> 4.479e-08","GO:0051169 (356) <br> nuclear transport <br> 5.217e-08","GO:0006996 (4136) <br> organelle organization <br> 7.280e-08","GO:0031503 (289) <br> protein-containing complex localization <br> 8.869e-08","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 1.827e-07","GO:0009266 (229) <br> response to temperature stimulus <br> 3.540e-07","GO:0015931 (246) <br> nucleobase-containing compound transport <br> 4.574e-07","GO:0051641 (3519) <br> cellular localization <br> 7.239e-07","GO:0019058 (386) <br> viral life cycle <br> 1.644e-06","GO:0044085 (3377) <br> cellular component biogenesis <br> 2.137e-06","GO:0006401 (431) <br> RNA catabolic process <br> 3.102e-06","GO:0010629 (1470) <br> negative regulation of gene expression <br> 5.036e-06","GO:0033036 (3148) <br> macromolecule localization <br> 5.757e-06","GO:0006090 (152) <br> pyruvate metabolic process <br> 6.029e-06","GO:0016043 (6661) <br> cellular component organization <br> 6.903e-06","GO:0008104 (2670) <br> protein localization <br> 7.616e-06","GO:0007010 (1473) <br> cytoskeleton organization <br> 8.817e-06","GO:0045184 (1866) <br> establishment of protein localization <br> 8.928e-06","GO:0072594 (523) <br> establishment of protein localization to organelle <br> 1.064e-05","GO:0006096 (116) <br> glycolytic process <br> 1.132e-05","GO:0006757 (117) <br> ATP generation from ADP <br> 1.231e-05","GO:0006402 (388) <br> mRNA catabolic process <br> 1.369e-05","GO:0015031 (1754) <br> protein transport <br> 1.620e-05","GO:0071705 (2216) <br> nitrogen compound transport <br> 1.668e-05","GO:0016071 (880) <br> mRNA metabolic process <br> 1.693e-05","GO:0046031 (125) <br> ADP metabolic process <br> 2.340e-05","GO:0009056 (2779) <br> catabolic process <br> 2.853e-05","GO:0002181 (149) <br> cytoplasmic translation <br> 2.883e-05","GO:0022607 (3104) <br> cellular component assembly <br> 3.489e-05","GO:0006165 (135) <br> nucleoside diphosphate phosphorylation <br> 4.914e-05","GO:0046939 (137) <br> nucleotide phosphorylation <br> 5.658e-05","GO:0060968 (138) <br> regulation of gene silencing <br> 6.066e-05","GO:0009135 (138) <br> purine nucleoside diphosphate metabolic process <br> 6.066e-05","GO:0009179 (138) <br> purine ribonucleoside diphosphate metabolic process <br> 6.066e-05","GO:0006886 (995) <br> intracellular protein transport <br> 6.235e-05","GO:0071763 (74) <br> nuclear membrane organization <br> 6.365e-05","GO:0034613 (1842) <br> cellular protein localization <br> 6.379e-05","GO:0009185 (141) <br> ribonucleoside diphosphate metabolic process <br> 7.451e-05","GO:0006413 (203) <br> translational initiation <br> 7.486e-05","GO:0070727 (1856) <br> cellular macromolecule localization <br> 7.862e-05","GO:0051649 (2778) <br> establishment of localization in cell <br> 8.259e-05","GO:0010608 (928) <br> posttranscriptional regulation of gene expression <br> 9.039e-05","GO:0016925 (78) <br> protein sumoylation <br> 1.034e-04","GO:0006412 (804) <br> translation <br> 1.128e-04","GO:0006110 (81) <br> regulation of glycolytic process <br> 1.395e-04","GO:0033365 (974) <br> protein localization to organelle <br> 1.554e-04","GO:0006606 (154) <br> protein import into nucleus <br> 1.721e-04","GO:0060964 (117) <br> regulation of gene silencing by miRNA <br> 1.835e-04","GO:0006998 (85) <br> nuclear envelope organization <br> 1.913e-04","GO:0060147 (119) <br> regulation of posttranscriptional gene silencing <br> 2.125e-04","GO:0007049 (1867) <br> cell cycle <br> 2.242e-04","GO:0022402 (1388) <br> cell cycle process <br> 2.242e-04","GO:0060966 (120) <br> regulation of gene silencing by RNA <br> 2.284e-04","GO:0043043 (834) <br> peptide biosynthetic process <br> 2.323e-04","GO:0009132 (160) <br> nucleoside diphosphate metabolic process <br> 2.468e-04","GO:0051170 (174) <br> import into nucleus <br> 3.227e-04","GO:0043470 (92) <br> regulation of carbohydrate catabolic process <br> 3.797e-04","GO:0014074 (153) <br> response to purine-containing compound <br> 4.208e-04","GO:0034655 (573) <br> nucleobase-containing compound catabolic process <br> 4.221e-04","GO:0033554 (2142) <br> cellular response to stress <br> 4.691e-04","GO:0071702 (2657) <br> organic substance transport <br> 5.629e-04","GO:0002934 (10) <br> desmosome organization <br> 5.668e-04","GO:1901575 (2289) <br> organic substance catabolic process <br> 8.479e-04","GO:0034330 (727) <br> cell junction organization <br> 8.798e-04","GO:0035973 (31) <br> aggrephagy <br> 1.151e-03","GO:0043604 (975) <br> amide biosynthetic process <br> 1.154e-03","GO:0099188 (14) <br> postsynaptic cytoskeleton organization <br> 1.247e-03","GO:0000184 (123) <br> nuclear-transcribed mRNA catabolic process, nonsense-mediated decay <br> 1.249e-03","GO:0046700 (617) <br> heterocycle catabolic process <br> 1.416e-03","GO:0009628 (1264) <br> response to abiotic stimulus <br> 1.569e-03","GO:0044270 (623) <br> cellular nitrogen compound catabolic process <br> 1.655e-03","GO:0034641 (6576) <br> cellular nitrogen compound metabolic process <br> 1.906e-03","GO:0016052 (201) <br> carbohydrate catabolic process <br> 2.039e-03","GO:0006518 (998) <br> peptide metabolic process <br> 2.124e-03","GO:0101024 (51) <br> mitotic nuclear membrane organization <br> 2.186e-03","GO:0007084 (51) <br> mitotic nuclear membrane reassembly <br> 2.186e-03","GO:1903047 (956) <br> mitotic cell cycle process <br> 2.238e-03","GO:0019439 (636) <br> aromatic compound catabolic process <br> 2.306e-03","GO:0034504 (294) <br> protein localization to nucleus <br> 2.715e-03","GO:0000278 (1098) <br> mitotic cell cycle <br> 2.922e-03","GO:1903578 (120) <br> regulation of ATP metabolic process <br> 2.935e-03","GO:0031468 (54) <br> nuclear membrane reassembly <br> 3.085e-03","GO:1900542 (121) <br> regulation of purine nucleotide metabolic process <br> 3.125e-03","GO:0009894 (1081) <br> regulation of catabolic process <br> 3.456e-03","GO:0006140 (123) <br> regulation of nucleotide metabolic process <br> 3.537e-03","GO:0043467 (167) <br> regulation of generation of precursor metabolites and energy <br> 3.759e-03","GO:0017038 (219) <br> protein import <br> 4.431e-03","GO:1901361 (664) <br> organic cyclic compound catabolic process <br> 4.566e-03","GO:0045109 (24) <br> intermediate filament organization <br> 5.098e-03","GO:0010605 (3262) <br> negative regulation of macromolecule metabolic process <br> 5.933e-03","GO:0000956 (214) <br> nuclear-transcribed mRNA catabolic process <br> 6.291e-03","GO:0097435 (863) <br> supramolecular fiber organization <br> 8.829e-03","GO:0086073 (6) <br> bundle of His cell-Purkinje myocyte adhesion involved in cell communication <br> 8.946e-03","GO:0034063 (26) <br> stress granule assembly <br> 8.963e-03","GO:0055086 (628) <br> nucleobase-containing small molecule metabolic process <br> 9.100e-03","GO:0051179 (6938) <br> localization <br> 9.224e-03","GO:0009892 (3498) <br> negative regulation of metabolic process <br> 9.756e-03","GO:0043603 (1284) <br> cellular amide metabolic process <br> 1.005e-02","GO:0046483 (5929) <br> heterocycle metabolic process <br> 1.075e-02","GO:0010467 (6192) <br> gene expression <br> 1.116e-02","GO:0006139 (5752) <br> nucleobase-containing compound metabolic process <br> 1.155e-02","GO:0006396 (1010) <br> RNA processing <br> 1.178e-02","GO:0016575 (84) <br> histone deacetylation <br> 1.225e-02","GO:0022613 (473) <br> ribonucleoprotein complex biogenesis <br> 1.264e-02","GO:0006997 (148) <br> nucleus organization <br> 1.324e-02","GO:0009259 (433) <br> ribonucleotide metabolic process <br> 1.361e-02","GO:0072657 (698) <br> protein localization to membrane <br> 1.380e-02","GO:0006614 (96) <br> SRP-dependent cotranslational protein targeting to membrane <br> 1.483e-02","GO:0044248 (2381) <br> cellular catabolic process <br> 1.538e-02","GO:0045110 (7) <br> intermediate filament bundle assembly <br> 1.559e-02","GO:0086042 (7) <br> cardiac muscle cell-cardiac muscle cell adhesion <br> 1.559e-02","GO:0043933 (2017) <br> protein-containing complex subunit organization <br> 1.673e-02","GO:0019693 (444) <br> ribose phosphate metabolic process <br> 1.781e-02","GO:0010468 (5218) <br> regulation of gene expression <br> 1.857e-02","GO:0046034 (317) <br> ATP metabolic process <br> 1.898e-02","GO:0032781 (60) <br> positive regulation of ATPase activity <br> 1.983e-02","GO:0006613 (102) <br> cotranslational protein targeting to membrane <br> 2.215e-02","GO:0098974 (12) <br> postsynaptic actin cytoskeleton organization <br> 2.362e-02","GO:0016458 (534) <br> gene silencing <br> 2.749e-02","GO:0006109 (215) <br> regulation of carbohydrate metabolic process <br> 2.938e-02","GO:0007052 (194) <br> mitotic spindle organization <br> 2.946e-02","GO:0070972 (144) <br> protein localization to endoplasmic reticulum <br> 3.028e-02","GO:0048519 (5861) <br> negative regulation of biological process <br> 3.035e-02","GO:0072521 (468) <br> purine-containing compound metabolic process <br> 3.115e-02","GO:0018205 (408) <br> peptidyl-lysine modification <br> 3.543e-02","GO:0072331 (290) <br> signal transduction by p53 class mediator <br> 3.593e-02","GO:0042274 (74) <br> ribosomal small subunit biogenesis <br> 3.732e-02","GO:0014075 (49) <br> response to amine <br> 3.835e-02","GO:0045047 (111) <br> protein targeting to ER <br> 3.855e-02","GO:0019240 (5) <br> citrulline biosynthetic process <br> 4.112e-02","GO:0000056 (8) <br> ribosomal small subunit export from nucleus <br> 4.194e-02","GO:0016446 (12) <br> somatic hypermutation of immunoglobulin genes <br> 4.585e-02","GO:0009150 (416) <br> purine ribonucleotide metabolic process <br> 4.670e-02","GO:0006476 (103) <br> protein deacetylation <br> 4.707e-02","GO:0097264 (8) <br> self proteolysis <br> 4.815e-02","GO:0051234 (5364) <br> establishment of localization <br> 4.828e-02","GO:0032386 (365) <br> regulation of intracellular transport <br> 4.843e-02","GO:0072599 (115) <br> establishment of protein localization to endoplasmic reticulum <br> 4.853e-02"],"key":["GO:0016032","GO:0071166","GO:0019080","GO:0071426","GO:0019083","GO:0051168","GO:0006405","GO:0071431","GO:0006409","GO:0006406","GO:0071427","GO:0009408","GO:0046907","GO:0051031","GO:0097064","GO:0050658","GO:0050657","GO:0051236","GO:0051028","GO:0075733","GO:0006403","GO:0034605","GO:0046794","GO:1900034","GO:0045104","GO:0044766","GO:1902579","GO:0045103","GO:0006913","GO:0051169","GO:0006996","GO:0031503","GO:0071840","GO:0009266","GO:0015931","GO:0051641","GO:0019058","GO:0044085","GO:0006401","GO:0010629","GO:0033036","GO:0006090","GO:0016043","GO:0008104","GO:0007010","GO:0045184","GO:0072594","GO:0006096","GO:0006757","GO:0006402","GO:0015031","GO:0071705","GO:0016071","GO:0046031","GO:0009056","GO:0002181","GO:0022607","GO:0006165","GO:0046939","GO:0060968","GO:0009135","GO:0009179","GO:0006886","GO:0071763","GO:0034613","GO:0009185","GO:0006413","GO:0070727","GO:0051649","GO:0010608","GO:0016925","GO:0006412","GO:0006110","GO:0033365","GO:0006606","GO:0060964","GO:0006998","GO:0060147","GO:0007049","GO:0022402","GO:0060966","GO:0043043","GO:0009132","GO:0051170","GO:0043470","GO:0014074","GO:0034655","GO:0033554","GO:0071702","GO:0002934","GO:1901575","GO:0034330","GO:0035973","GO:0043604","GO:0099188","GO:0000184","GO:0046700","GO:0009628","GO:0044270","GO:0034641","GO:0016052","GO:0006518","GO:0101024","GO:0007084","GO:1903047","GO:0019439","GO:0034504","GO:0000278","GO:1903578","GO:0031468","GO:1900542","GO:0009894","GO:0006140","GO:0043467","GO:0017038","GO:1901361","GO:0045109","GO:0010605","GO:0000956","GO:0097435","GO:0086073","GO:0034063","GO:0055086","GO:0051179","GO:0009892","GO:0043603","GO:0046483","GO:0010467","GO:0006139","GO:0006396","GO:0016575","GO:0022613","GO:0006997","GO:0009259","GO:0072657","GO:0006614","GO:0044248","GO:0045110","GO:0086042","GO:0043933","GO:0019693","GO:0010468","GO:0046034","GO:0032781","GO:0006613","GO:0098974","GO:0016458","GO:0006109","GO:0007052","GO:0070972","GO:0048519","GO:0072521","GO:0018205","GO:0072331","GO:0042274","GO:0014075","GO:0045047","GO:0019240","GO:0000056","GO:0016446","GO:0009150","GO:0006476","GO:0097264","GO:0051234","GO:0032386","GO:0072599"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.5761024288984,16.7265158057432,17.613817251315,16.7146488256242,17.2176438847111,17.380365289039,16.7026791840164,14.4360075836772,14.4360075836772,16.481642870719,16.481642870719,17.0229653267471,20.2385834258228,14.5938600426124,14.6431687998088,17.2340225412223,17.2340225412223,17.2582214035679,16.9446117361575,15.3670429037096,17.5187744904594,16.5769367992576,15.5656749705695,15.9184363351098,15.1381652632347,15.6178829039023,15.6178829039023,15.1730510425948,18.1794066032068,18.191179668157,21.2723467043792,17.8982555070379,21.8459599395529,17.5639837899912,17.6677052476822,21.0862719030328,18.3032532553118,21.0385096214352,18.4546244176415,20.044069731458,20.9567768885572,16.9546559960072,21.8100397125636,20.7635451231059,20.0465816412379,20.3353801650254,18.7165034152978,16.5368885741886,16.5503659342062,18.3103826663661,20.2602817247612,20.5421825061326,19.399049958471,16.6537390526195,20.810694186256,16.9242986933942,20.9403466721754,16.772989997304,16.7956528052537,16.8068458716563,16.8068458716563,16.8068458716563,19.5558507009536,15.8100072347003,20.319702550628,16.8398886539314,17.3876399430454,20.3288742747175,20.8102706711732,19.4670456085751,15.8973863612294,19.2827264956164,15.9596427433532,19.5287296832397,16.9745252158235,16.5503659342062,16.0386970905277,16.5769367992576,20.3360286952706,19.973189582811,16.5900349098891,19.3300112439233,17.0324495724136,17.1586931132012,16.1673930557028,16.9646268148802,18.8385081160232,20.5015270643292,20.7577848508211,11.8112973047217,20.5808878062642,19.1520361624938,14.2608475416207,19.5300354328683,12.6110489969071,16.6286063334144,18.9366475606211,19.856956548733,18.9494378784552,21.7957590763696,17.3730506102813,19.5596745798845,15.1730510425948,15.1730510425948,19.5049752239369,18.9766967766795,17.9225826332061,19.6804883602879,16.5900349098891,15.2731516632256,16.603011108722,19.660809759004,16.6286063334144,17.0970441678342,17.4989140574595,19.0334065446586,13.7594961631324,20.9982306119388,17.4651348098225,19.3740007383699,10.4153745295623,13.9189573720891,18.9599946027491,21.8552699322811,21.0793394170726,19.876513767642,21.6801776076722,21.7287038516454,21.646213660555,19.5748468722174,16.0193351590306,18.5810671803456,16.9140277546203,18.4609454905787,19.0988745407174,16.2360552317553,20.6278258240816,10.8656489767313,10.8656489767313,20.4293081746705,18.4951505633078,21.5365868696029,18.0289154675614,15.45541914919,16.3332135329498,12.254025725742,18.7444040158983,17.4719600731541,17.3206905810587,16.8721549697602,21.6672583922471,18.5666713375747,18.3795391978703,17.90315779907,15.8100072347003,15.1024639120198,16.4674845293838,9.83978094370953,11.2337416908317,12.254025725742,18.4061694695621,16.3487788308752,11.2337416908317,21.5677047149222,18.2258567164575,16.5232800867865],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[55.6531310804885,59.9463115386647,64.4346365631215,62.1475422463113,59.4623530142884,62.1280277896833,59.4857703622421,62.1436393549857,62.1319306810089,55.3213853178112,54.9701250985059,54.9271932939242,71.15541542583,69.2781246982094,62.3582983778945,57.4523639815969,62.1397364636601,55.0013482291108,64.6102666727742,62.120222007032,68.4546146285047,55.0364742510414,68.5756042595987,68.4585175198303,57.8270415488559,68.5795071509243,55.9653623865377,55.738994689652,55.9185276906303,62.1241248983576,57.8231386575303,55.9146247993047,69.434240351234,54.9740279898315,60.4927163242507,63.3223125353214,57.7802068529485,70.4841181178243,61.9875237019611,57.9558369626012,68.8839326743223,58.0729237023696,60.2507370620626,55.8131496248387,57.2064818280831,65.2191177195701,57.1362297842221,62.9281205114343,55.7663149289313,55.6921599937446,57.8348473315071,57.0074343704768,58.1080497243001,55.2940650785319,64.5244030636107,59.4896732535677,57.1596471321758,61.9914265932867,56.0082941911194,69.6488993741428,62.1358335723345,64.4307336717959,66.2728983774861,57.2845396545954,67.9160156255698,59.7394582984071,55.789732276885,55.7233831243496,57.6553143305288,56.9996285878256,63.1388766430175,55.5945877106043,58.1939133334637,57.9441282886243,63.7438247984877,54.4510405521992,58.0143803324854,58.7403181190497,68.6653707600879],"y":[17,17,12.6899687498077,12.6899687498077,12.6899687498077,12.6203625331755,12.5382150458087,12.1285633583622,11.2989560523466,11.0359398497754,10.3640591332277,10.2600186577425,9.17995750665603,9.04519904180791,9.00427223731318,9.00427223731318,8.79384955418534,8.68526707837668,7.43456996650501,7.39530946930525,7.23946278833177,7.138583069669,6.98313237446696,6.91528215301861,6.48901027189364,5.83407490313635,5.74417111674199,5.24564513297533,4.68647678784396,4.48974127388769,4.47822737891405,4.18945976426546,3.99140164425646,3.98072009563064,3.92814615660617,3.70427738981851,3.56505191489969,3.08863712426412,2.97338188385964,2.85891209936763,2.84070973742931,2.81027893392827,2.80861202979472,2.77444349325753,2.61455859807018,2.55699460316144,2.55621766866248,2.55335563921182,2.47610870193325,2.42057280321818,2.38728868211313,2.28546912573102,2.2616179425312,2.23314628086193,2.21069414605163,2.10773942902419,2.0285541892457,2.01769160831063,1.9818921790297,1.85359422249694,1.84547908651055,1.84547908651055,1.79502076356101,1.79502076356101,1.75011844577879,1.71023744271782,1.66148605638286,1.66143959506091,1.62484004532006,1.61513571164753,1.60528034089371,1.5289081874249,1.51052222044942,1.44302544432001,1.44302544432001,1.39380792486525,1.32243185271358,1.32207945228188,1.31575919377388],"text":["GO:0005829 (5361) <br> cytosol <br> 1.396e-17","GO:0032991 (5541) <br> protein-containing complex <br> 2.448e-17","GO:0070013 (5579) <br> intracellular organelle lumen <br> 2.042e-13","GO:0043233 (5579) <br> organelle lumen <br> 2.042e-13","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 2.042e-13","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 2.397e-13","GO:0031981 (4453) <br> nuclear lumen <br> 2.896e-13","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 7.438e-13","GO:0043229 (13117) <br> intracellular organelle <br> 5.024e-12","GO:0005737 (11951) <br> cytoplasm <br> 9.206e-12","GO:0005634 (7607) <br> nucleus <br> 4.325e-11","GO:0005622 (14808) <br> intracellular anatomical structure <br> 5.495e-11","GO:1990904 (694) <br> ribonucleoprotein complex <br> 6.608e-10","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 9.012e-10","GO:0043657 (58) <br> host cell <br> 9.902e-10","GO:0018995 (58) <br> host cellular component <br> 9.902e-10","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 1.607e-09","GO:0005643 (85) <br> nuclear pore <br> 2.064e-09","GO:0070161 (838) <br> anchoring junction <br> 3.676e-08","GO:0043226 (14027) <br> organelle <br> 4.024e-08","GO:0099080 (1325) <br> supramolecular complex <br> 5.762e-08","GO:0005654 (4112) <br> nucleoplasm <br> 7.268e-08","GO:0099512 (997) <br> supramolecular fiber <br> 1.040e-07","GO:0099081 (1005) <br> supramolecular polymer <br> 1.215e-07","GO:0030055 (427) <br> cell-substrate junction <br> 3.243e-07","GO:0099513 (766) <br> polymeric cytoskeletal fiber <br> 1.465e-06","GO:0005925 (420) <br> focal adhesion <br> 1.802e-06","GO:0005856 (2339) <br> cytoskeleton <br> 5.680e-06","GO:0005912 (171) <br> adherens junction <br> 2.058e-05","GO:0043227 (13201) <br> membrane-bounded organelle <br> 3.238e-05","GO:0030054 (2107) <br> cell junction <br> 3.325e-05","GO:0005911 (496) <br> cell-cell junction <br> 6.465e-05","GO:1902494 (1377) <br> catalytic complex <br> 1.020e-04","GO:0005635 (471) <br> nuclear envelope <br> 1.045e-04","GO:0034399 (132) <br> nuclear periphery <br> 1.180e-04","GO:0045111 (260) <br> intermediate filament cytoskeleton <br> 1.976e-04","GO:0022626 (106) <br> cytosolic ribosome <br> 2.722e-04","GO:1990391 (36) <br> DNA repair complex <br> 8.154e-04","GO:0042995 (2337) <br> cell projection <br> 1.063e-03","GO:0030120 (55) <br> vesicle coat <br> 1.384e-03","GO:0120025 (2234) <br> plasma membrane bounded cell projection <br> 1.443e-03","GO:0030286 (55) <br> dynein complex <br> 1.548e-03","GO:0033290 (15) <br> eukaryotic 48S preinitiation complex <br> 1.554e-03","GO:0005882 (219) <br> intermediate filament <br> 1.681e-03","GO:0016363 (110) <br> nuclear matrix <br> 2.429e-03","GO:0070993 (18) <br> translation preinitiation complex <br> 2.773e-03","GO:0016020 (9829) <br> membrane <br> 2.778e-03","GO:0044391 (191) <br> ribosomal subunit <br> 2.797e-03","GO:0005868 (21) <br> cytoplasmic dynein complex <br> 3.341e-03","GO:0005840 (247) <br> ribosome <br> 3.797e-03","GO:0030057 (25) <br> desmosome <br> 4.099e-03","GO:0012505 (4651) <br> endomembrane system <br> 5.182e-03","GO:0030424 (661) <br> axon <br> 5.475e-03","GO:0005730 (955) <br> nucleolus <br> 5.846e-03","GO:0070062 (2178) <br> extracellular exosome <br> 6.156e-03","GO:0031982 (4058) <br> vesicle <br> 7.803e-03","GO:0016222 (2) <br> procollagen-proline 4-dioxygenase complex <br> 9.364e-03","GO:0043005 (1384) <br> neuron projection <br> 9.601e-03","GO:0005938 (326) <br> cell cortex <br> 1.043e-02","GO:1903561 (2263) <br> extracellular vesicle <br> 1.401e-02","GO:0043230 (2265) <br> extracellular organelle <br> 1.427e-02","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 1.427e-02","GO:0090545 (16) <br> CHD-type complex <br> 1.603e-02","GO:0016581 (16) <br> NuRD complex <br> 1.603e-02","GO:0098687 (348) <br> chromosomal region <br> 1.778e-02","GO:0032301 (3) <br> MutSalpha complex <br> 1.949e-02","GO:0005874 (435) <br> microtubule <br> 2.180e-02","GO:0005851 (6) <br> eukaryotic translation initiation factor 2B complex <br> 2.181e-02","GO:0019897 (178) <br> extrinsic component of plasma membrane <br> 2.372e-02","GO:0010494 (77) <br> cytoplasmic stress granule <br> 2.426e-02","GO:0044611 (3) <br> nuclear pore inner ring <br> 2.482e-02","GO:0005813 (620) <br> centrosome <br> 2.959e-02","GO:0030662 (177) <br> coated vesicle membrane <br> 3.087e-02","GO:0030117 (97) <br> membrane coat <br> 3.606e-02","GO:0048475 (97) <br> coated membrane <br> 3.606e-02","GO:0001533 (44) <br> cornified envelope <br> 4.038e-02","GO:0030135 (296) <br> coated vesicle <br> 4.760e-02","GO:0031252 (426) <br> cell leading edge <br> 4.763e-02","GO:0101031 (27) <br> chaperone complex <br> 4.833e-02"],"key":["GO:0005829","GO:0032991","GO:0070013","GO:0043233","GO:0031974","GO:0043228","GO:0031981","GO:0043232","GO:0043229","GO:0005737","GO:0005634","GO:0005622","GO:1990904","GO:0140513","GO:0043657","GO:0018995","GO:0043231","GO:0005643","GO:0070161","GO:0043226","GO:0099080","GO:0005654","GO:0099512","GO:0099081","GO:0030055","GO:0099513","GO:0005925","GO:0005856","GO:0005912","GO:0043227","GO:0030054","GO:0005911","GO:1902494","GO:0005635","GO:0034399","GO:0045111","GO:0022626","GO:1990391","GO:0042995","GO:0030120","GO:0120025","GO:0030286","GO:0033290","GO:0005882","GO:0016363","GO:0070993","GO:0016020","GO:0044391","GO:0005868","GO:0005840","GO:0030057","GO:0012505","GO:0030424","GO:0005730","GO:0070062","GO:0031982","GO:0016222","GO:0043005","GO:0005938","GO:1903561","GO:0043230","GO:0065010","GO:0090545","GO:0016581","GO:0098687","GO:0032301","GO:0005874","GO:0005851","GO:0019897","GO:0010494","GO:0044611","GO:0005813","GO:0030662","GO:0030117","GO:0048475","GO:0001533","GO:0030135","GO:0031252","GO:0101031"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[21.5670744132122,21.604243722507,21.6119263993467,21.6119263993467,21.6119263993467,21.5111955995147,21.3567493317634,21.5094218975349,22.5481301823792,22.4484604312869,21.9570470414316,22.6771653543307,19.0913542161997,19.8290088862669,15.3970834943308,15.3970834943308,22.4569580819102,16.0386970905277,19.3361763823889,22.6196160803639,19.91559986261,21.2656780982146,19.5584013351635,19.568548877415,18.4418855248262,19.2200109729712,18.41927534827,20.6066429891266,17.1326162234276,22.5549451746932,20.4817706635075,18.6452157443254,19.9633396074552,18.5753288674768,16.7382817982581,17.7473483032527,16.394472388579,14.5429649371964,20.6056241461277,15.3051018762464,20.5518518073364,15.3051018762464,12.766247544801,17.4989140574595,16.4531820242158,13.1640323425788,22.2374238416628,17.2976054697329,13.4876469321386,17.6735580678764,13.8411375859653,21.4062815734808,19.0274559713263,19.5036413861221,20.5214902439301,21.250520850314,3.77952755905512,19.9696175275459,18.0682354374322,20.5672559227953,20.5683104706776,20.5683104706776,12.9090380872033,12.9090380872033,18.1595388624734,7.81088103584182,18.4672347324523,10.4153745295623,17.1926950579195,15.8760274247573,7.81088103584182,18.9430595403157,17.1842747871435,16.2527164787856,16.2527164787856,14.9103337480098,17.9321863039161,18.4386804137039,13.9932800960407],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[3.70531986857705,37.4331908619671,29.9914403141949,3.55312885055759,29.8158352934032,48.6250841870906,42.2096474275009,17.3908044889425,8.6066511155628,9.3285828677064,20.0170751332271,9.77344892037868,19.6931814282113,3.7521478741215,41.83112104935,21.6521529934875,29.9446123086505,4.89943400996052,41.7725860424194,39.474111436946,21.0511935890003,9.64076957133607,42.6232948098103,9.36370387186474,49.3431136054388,34.7210688741843,22.9008998080061,42.6193924760149,2.41754971610468,42.8340208347603,23.1116258329561,30.1280219970329,41.7608790410333,21.8355626818699,3.74824554032613,26.9281082848288,2.13267934904261,48.6133771857045,37.5619678772143,22.8969974742107,22.8774858052339,42.8301185009649,17.027887445973,38.0575642692264,17.4454371620777,42.7481694912621,3.57264051953444,49.7411516525666],"y":[17,9.85364481706043,9.19226822783498,9.02272928400442,8.16088138523119,7.74221253899359,7.37444077691774,7.18319771903359,6.26377948486804,5.72972083689664,4.56754589830854,4.19808860822732,4.05262533857352,3.87581766044037,3.63739889079394,3.11237025798423,3.06428250899763,2.98504314162909,2.96076055210354,2.75037580371492,2.74904505776605,2.33556243834871,2.23495943363648,2.11224150435564,2.10103212719438,1.97668906885308,1.93403912131409,1.87266696980287,1.86600414833818,1.86375311966821,1.81388293873631,1.78377031768664,1.75448517697966,1.75173478235155,1.73963413809767,1.70409780896207,1.70279637167631,1.69901470370372,1.56735109351738,1.45822198527433,1.45822198527433,1.39940344392084,1.36838400256081,1.36769112620924,1.3672912973837,1.35052672150408,1.34678644754536,1.3420353554229],"text":["GO:0003723 (1951) <br> RNA binding <br> 5.833e-18","GO:0050839 (544) <br> cell adhesion molecule binding <br> 1.401e-10","GO:0045296 (333) <br> cadherin binding <br> 6.423e-10","GO:0003676 (4252) <br> nucleic acid binding <br> 9.490e-10","GO:0044877 (1304) <br> protein-containing complex binding <br> 6.904e-09","GO:1901363 (6210) <br> heterocyclic compound binding <br> 1.810e-08","GO:0097159 (6291) <br> organic cyclic compound binding <br> 4.222e-08","GO:0017056 (28) <br> structural constituent of nuclear pore <br> 6.558e-08","GO:0005198 (701) <br> structural molecule activity <br> 5.448e-07","GO:0005515 (14767) <br> protein binding <br> 1.863e-06","GO:0019899 (2048) <br> enzyme binding <br> 2.707e-05","GO:0008135 (99) <br> translation factor activity, RNA binding <br> 6.337e-05","GO:0019215 (15) <br> intermediate filament binding <br> 8.859e-05","GO:0003743 (58) <br> translation initiation factor activity <br> 1.331e-04","GO:0090079 (123) <br> translation regulator activity, nucleic acid binding <br> 2.305e-04","GO:0031072 (127) <br> heat shock protein binding <br> 7.720e-04","GO:0045182 (154) <br> translation regulator activity <br> 8.624e-04","GO:0004088 (2) <br> carbamoyl-phosphate synthase (glutamine-hydrolyzing) activity <br> 1.035e-03","GO:0086083 (5) <br> cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication <br> 1.095e-03","GO:0060590 (43) <br> ATPase regulator activity <br> 1.777e-03","GO:0030554 (1577) <br> adenyl nucleotide binding <br> 1.782e-03","GO:0008092 (1001) <br> cytoskeletal protein binding <br> 4.618e-03","GO:0098632 (49) <br> cell-cell adhesion mediator activity <br> 5.822e-03","GO:0005524 (1500) <br> ATP binding <br> 7.723e-03","GO:1990254 (6) <br> keratin filament binding <br> 7.924e-03","GO:0048156 (45) <br> tau protein binding <br> 1.055e-02","GO:0032143 (2) <br> single thymine insertion binding <br> 1.164e-02","GO:0098631 (58) <br> cell adhesion mediator activity <br> 1.341e-02","GO:0000774 (11) <br> adenyl-nucleotide exchange factor activity <br> 1.361e-02","GO:0099186 (12) <br> structural constituent of postsynapse <br> 1.369e-02","GO:0032559 (1564) <br> adenyl ribonucleotide binding <br> 1.535e-02","GO:0045505 (29) <br> dynein intermediate chain binding <br> 1.645e-02","GO:0086080 (11) <br> protein binding involved in heterotypic cell-cell adhesion <br> 1.760e-02","GO:0031492 (39) <br> nucleosomal DNA binding <br> 1.771e-02","GO:0003735 (171) <br> structural constituent of ribosome <br> 1.821e-02","GO:0036094 (2516) <br> small molecule binding <br> 1.977e-02","GO:0000166 (2176) <br> nucleotide binding <br> 1.982e-02","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 2.000e-02","GO:0051087 (106) <br> chaperone binding <br> 2.708e-02","GO:0032142 (3) <br> single guanine insertion binding <br> 3.482e-02","GO:0032137 (3) <br> guanine/thymine mispair binding <br> 3.482e-02","GO:0099184 (3) <br> structural constituent of postsynaptic intermediate filament cytoskeleton <br> 3.987e-02","GO:0016887 (553) <br> ATPase <br> 4.282e-02","GO:0051879 (43) <br> Hsp90 protein binding <br> 4.289e-02","GO:0017076 (1932) <br> purine nucleotide binding <br> 4.292e-02","GO:0098973 (4) <br> structural constituent of postsynaptic actin cytoskeleton <br> 4.461e-02","GO:0003682 (573) <br> chromatin binding <br> 4.500e-02","GO:1990841 (58) <br> promoter-specific chromatin binding <br> 4.550e-02"],"key":["GO:0003723","GO:0050839","GO:0045296","GO:0003676","GO:0044877","GO:1901363","GO:0097159","GO:0017056","GO:0005198","GO:0005515","GO:0019899","GO:0008135","GO:0019215","GO:0003743","GO:0090079","GO:0031072","GO:0045182","GO:0004088","GO:0086083","GO:0060590","GO:0030554","GO:0008092","GO:0098632","GO:0005524","GO:1990254","GO:0048156","GO:0032143","GO:0098631","GO:0000774","GO:0099186","GO:0032559","GO:0045505","GO:0086080","GO:0031492","GO:0003735","GO:0036094","GO:0000166","GO:1901265","GO:0051087","GO:0032142","GO:0032137","GO:0099184","GO:0016887","GO:0051879","GO:0017076","GO:0098973","GO:0003682","GO:1990841"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[20.3892145517334,18.7692301298177,18.0980022367888,21.3040075767332,19.8957455135249,21.7319446023732,21.7464055517094,14.064391440917,19.1044841513478,22.6742247680722,20.4476570765472,16.2854648740137,12.766247544801,15.3970834943308,16.6286063334144,16.6784250132177,16.9745252158235,3.77952755905512,9.83978094370953,14.8688640092558,20.1304163003948,19.5634860592025,15.1024639120198,20.0689442128028,10.4153745295623,14.9507227107922,3.77952755905512,15.3970834943308,12.0456958427095,12.254025725742,20.1202673855808,14.1325441121217,12.0456958427095,14.69098244776,17.1326162234276,20.6932989274541,20.5203904843139,20.5209404994456,16.394472388579,7.81088103584182,7.81088103584182,7.81088103584182,18.791152293927,14.8688640092558,20.3774023473002,9.05045292005097,18.8385081160232,15.3970834943308],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[198.678691452357,199.339342146347,199.691167959715],"y":[3.56752470910146,2.27420802890387,2.20576282431003],"text":["KEGG:03013 (180) <br> RNA transport <br> 2.707e-04","KEGG:04962 (44) <br> Vasopressin-regulated water reabsorption <br> 5.319e-03","KEGG:05412 (77) <br> Arrhythmogenic right ventricular cardiomyopathy <br> 6.226e-03"],"key":["KEGG:03013","KEGG:04962","KEGG:05412"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(221,68,119,1)","opacity":0.8,"size":[17.209378989379,14.9103337480098,15.8760274247573],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(221,68,119,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"KEGG","legendgroup":"KEGG","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,49.9206590071537],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.8226750487752,71.2217645783655],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.123780619987,193.971385215695],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.873401257316,200.003902016042],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[25.9603295035768,62.5222198135703,134.547582917841,198.938651636679],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"44b832363c7c":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"44b822092d46":{"x":{},"xend":{},"y":{},"yend":{}},"44b8185c61d6":{"x":{},"xend":{},"y":{},"yend":{}},"44b8ce66273":{"x":{},"xend":{},"y":{},"yend":{}},"44b876c13e5b":{"x":{},"xend":{},"y":{},"yend":{}},"44b8753b9ae":{"x":{},"y":{}},"44b82c017b86":{"yintercept":{}}},"cur_data":"44b832363c7c","visdat":{"44b832363c7c":["function (y) ","x"],"44b822092d46":["function (y) ","x"],"44b8185c61d6":["function (y) ","x"],"44b8ce66273":["function (y) ","x"],"44b876c13e5b":["function (y) ","x"],"44b8753b9ae":["function (y) ","x"],"44b82c017b86":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData2a0dd1ba"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))
}</code></pre>
<div id="htmlwidget-e4a421697d58abce2053" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-e4a421697d58abce2053">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296"],["GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","KEGG","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","KEGG","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","KEGG","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:MF","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP"],[438,470,1570,2720,2134,1446,2129,1452,5659,2133,19292,6459,2130,385,19510,295,6462,284,9081,15683,2409,7174,4442,3961,399,2188,931,19515,2413,2410,19511,2132,3926,14486,303,15572,21689,15295,15294,15740,15569,7129,20879,2407,10003,11949,14410,23356,13164,2765,13018,25721,2127,13163,10305,2866,15684,3750,3945,312,2933,8362,3781,3751,19815,1027,3876,5598,1694,16046,3782,6440,550,1879,12668,2405,4787,492,9231,2130,5665,3542,2946,13203,20407,2133,2733,2406,5363,19740,5688,538,13775,4618,3672,859,2128,1026,7622,2194,14500,17770,3747,3790,2842,1993,19766,10009,537,3796,2416,19079,16054,4535,4767,4001,5882,296,2415,1710,450,2147,9428,2591,17766,2935,2435,16996,2976,7600,17768,10208,12141,3744,207,1015,15685,12386,5203,10037,9546,19737,1575,5037,4270,24834,7162,9873,744,2093,10193,10822,12473,22367,75,1060,14349,3860,1090,1648,4009,12746,511,9604,4883,10027,5672,2505,22537,3007,26133,6697,868,9987,2921,850,2334,118,26605,8356,23844,499,4227,2174,12383,480,1029,5901,24637,1959,13169,817,376,1099,10411,378,4764,2743,466,332,1888,1453,12133,21897,21066,9678,16839,15694,856,2094,4225,12472,561,8386,14204,4661,2173,5357,2400,5872,7628,2934,10410,3869,108,10465,20427,4056,2131,2719,2599,5411,12732,13170,21043,3191,888,7209,12553,10190,5084,3612,6935,449,4662,13778,1517,6389,35,9049,11946,505,488,2598,22257,983,815,2388,9114,5834,2146,2978,455,19228,14886,1121,20371,5356,5351,6107,20252,1057,2543,11682,5204,13134,10464,162,6508,27,3852,9241,3959,10443,404,12235,5831,3762,2466,1075,1261,21787,15738,3804,8759,20412],["RNA binding","cytosol","protein-containing complex","intracellular organelle lumen","organelle lumen","membrane-enclosed lumen","non-membrane-bounded organelle","nuclear lumen","viral process","intracellular non-membrane-bounded organelle","ribonucleoprotein complex localization","viral gene expression","intracellular organelle","cytoplasm","ribonucleoprotein complex export from nucleus","nucleus","viral transcription","intracellular anatomical structure","cell adhesion molecule binding","nuclear export","RNA export from nucleus","cadherin binding","ribonucleoprotein complex","nuclear protein-containing complex","nucleic acid binding","host cell","host cellular component","tRNA-containing ribonucleoprotein complex export from nucleus","tRNA export from nucleus","mRNA export from nucleus","mRNA-containing ribonucleoprotein complex export from nucleus","intracellular membrane-bounded organelle","response to heat","intracellular transport","nuclear pore","tRNA transport","ncRNA export from nucleus","RNA transport","nucleic acid transport","establishment of RNA localization","mRNA transport","protein-containing complex binding","intracellular transport of virus","RNA localization","cellular response to heat","heterocyclic compound binding","transport of virus","regulation of cellular response to heat","intermediate filament cytoskeleton organization","anchoring junction","multi-organism transport","multi-organism localization","organelle","intermediate filament-based process","organic cyclic compound binding","nucleocytoplasmic transport","nuclear transport","supramolecular complex","structural constituent of nuclear pore","nucleoplasm","organelle organization","protein-containing complex localization","supramolecular fiber","supramolecular polymer","cellular component organization or biogenesis","cell-substrate junction","response to temperature stimulus","nucleobase-containing compound transport","structural molecule activity","cellular localization","polymeric cytoskeletal fiber","viral life cycle","focal adhesion","protein binding","cellular component biogenesis","RNA catabolic process","negative regulation of gene expression","cytoskeleton","macromolecule localization","pyruvate metabolic process","cellular component organization","protein localization","cytoskeleton organization","establishment of protein localization","establishment of protein localization to organelle","glycolytic process","ATP generation from ADP","mRNA catabolic process","protein transport","nitrogen compound transport","mRNA metabolic process","adherens junction","ADP metabolic process","enzyme binding","catabolic process","cytoplasmic translation","membrane-bounded organelle","cell junction","cellular component assembly","nucleoside diphosphate phosphorylation","nucleotide phosphorylation","regulation of gene silencing","purine nucleoside diphosphate metabolic process","purine ribonucleoside diphosphate metabolic process","intracellular protein transport","translation factor activity, RNA binding","nuclear membrane organization","cellular protein localization","cell-cell junction","ribonucleoside diphosphate metabolic process","translational initiation","cellular macromolecule localization","establishment of localization in cell","intermediate filament binding","posttranscriptional regulation of gene expression","catalytic complex","protein sumoylation","nuclear envelope","translation","nuclear periphery","translation initiation factor activity","regulation of glycolytic process","protein localization to organelle","protein import into nucleus","regulation of gene silencing by miRNA","nuclear envelope organization","intermediate filament cytoskeleton","regulation of posttranscriptional gene silencing","cell cycle","cell cycle process","regulation of gene silencing by RNA","translation regulator activity, nucleic acid binding","peptide biosynthetic process","nucleoside diphosphate metabolic process","RNA transport","cytosolic ribosome","import into nucleus","regulation of carbohydrate catabolic process","response to purine-containing compound","nucleobase-containing compound catabolic process","cellular response to stress","organic substance transport","desmosome organization","heat shock protein binding","DNA repair complex","organic substance catabolic process","translation regulator activity","cell junction organization","carbamoyl-phosphate synthase (glutamine-hydrolyzing) activity","cell projection","cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication","aggrephagy","amide biosynthetic process","postsynaptic cytoskeleton organization","nuclear-transcribed mRNA catabolic process, nonsense-mediated decay","vesicle coat","heterocycle catabolic process","plasma membrane bounded cell projection","dynein complex","eukaryotic 48S preinitiation complex","response to abiotic stimulus","cellular nitrogen compound catabolic process","intermediate filament","ATPase regulator activity","adenyl nucleotide binding","cellular nitrogen compound metabolic process","carbohydrate catabolic process","peptide metabolic process","mitotic nuclear membrane organization","mitotic nuclear membrane reassembly","mitotic cell cycle process","aromatic compound catabolic process","nuclear matrix","protein localization to nucleus","translation preinitiation complex","membrane","ribosomal subunit","mitotic cell cycle","regulation of ATP metabolic process","nuclear membrane reassembly","regulation of purine nucleotide metabolic process","cytoplasmic dynein complex","regulation of catabolic process","regulation of nucleotide metabolic process","regulation of generation of precursor metabolites and energy","ribosome","desmosome","protein import","organic cyclic compound catabolic process","cytoskeletal protein binding","intermediate filament organization","endomembrane system","Vasopressin-regulated water reabsorption","axon","cell-cell adhesion mediator activity","nucleolus","negative regulation of macromolecule metabolic process","extracellular exosome","Arrhythmogenic right ventricular cardiomyopathy","nuclear-transcribed mRNA catabolic process","ATP binding","vesicle","keratin filament binding","supramolecular fiber organization","bundle of His cell-Purkinje myocyte adhesion involved in cell communication","stress granule assembly","nucleobase-containing small molecule metabolic process","localization","procollagen-proline 4-dioxygenase complex","neuron projection","negative regulation of metabolic process","cellular amide metabolic process","cell cortex","tau protein binding","heterocycle metabolic process","gene expression","nucleobase-containing compound metabolic process","single thymine insertion binding","RNA processing","histone deacetylation","ribonucleoprotein complex biogenesis","nucleus organization","cell adhesion mediator activity","ribonucleotide metabolic process","adenyl-nucleotide exchange factor activity","structural constituent of postsynapse","protein localization to membrane","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","SRP-dependent cotranslational protein targeting to membrane","adenyl ribonucleotide binding","cellular catabolic process","intermediate filament bundle assembly","cardiac muscle cell-cardiac muscle cell adhesion","CHD-type complex","NuRD complex","dynein intermediate chain binding","protein-containing complex subunit organization","protein binding involved in heterotypic cell-cell adhesion","nucleosomal DNA binding","chromosomal region","ribose phosphate metabolic process","structural constituent of ribosome","regulation of gene expression","ATP metabolic process","MutSalpha complex","small molecule binding","nucleotide binding","positive regulation of ATPase activity","nucleoside phosphate binding","microtubule","eukaryotic translation initiation factor 2B complex","cotranslational protein targeting to membrane","postsynaptic actin cytoskeleton organization","extrinsic component of plasma membrane","cytoplasmic stress granule","nuclear pore inner ring","chaperone binding","gene silencing","regulation of carbohydrate metabolic process","mitotic spindle organization","centrosome","protein localization to endoplasmic reticulum","negative regulation of biological process","coated vesicle membrane","purine-containing compound metabolic process","single guanine insertion binding","guanine/thymine mispair binding","peptidyl-lysine modification","signal transduction by p53 class mediator","membrane coat","coated membrane","ribosomal small subunit biogenesis","response to amine","protein targeting to ER","structural constituent of postsynaptic intermediate filament cytoskeleton","cornified envelope","citrulline biosynthetic process","ribosomal small subunit export from nucleus","ATPase","Hsp90 protein binding","purine nucleotide binding","structural constituent of postsynaptic actin cytoskeleton","chromatin binding","promoter-specific chromatin binding","somatic hypermutation of immunoglobulin genes","purine ribonucleotide metabolic process","protein deacetylation","coated vesicle","cell leading edge","self proteolysis","establishment of localization","chaperone complex","regulation of intracellular transport","establishment of protein localization to endoplasmic reticulum"],["5.8e-18","1.4e-17","2.4e-17","2.0e-13","2.0e-13","2.0e-13","2.4e-13","2.9e-13","4.9e-13","7.4e-13","8.6e-13","1.7e-12","5.0e-12","9.2e-12","1.4e-11","4.3e-11","4.9e-11","5.5e-11","1.4e-10","1.5e-10","2.1e-10","6.4e-10","6.6e-10","9.0e-10","9.5e-10","9.9e-10","9.9e-10","1.1e-09","1.1e-09","1.4e-09","1.4e-09","1.6e-09","1.9e-09","1.9e-09","2.1e-09","2.8e-09","3.8e-09","4.1e-09","4.1e-09","5.4e-09","6.1e-09","6.9e-09","8.3e-09","9.1e-09","1.0e-08","1.8e-08","2.8e-08","3.2e-08","3.4e-08","3.7e-08","3.9e-08","3.9e-08","4.0e-08","4.1e-08","4.2e-08","4.5e-08","5.2e-08","5.8e-08","6.6e-08","7.3e-08","7.3e-08","8.9e-08","1.0e-07","1.2e-07","1.8e-07","3.2e-07","3.5e-07","4.6e-07","5.4e-07","7.2e-07","1.5e-06","1.6e-06","1.8e-06","1.9e-06","2.1e-06","3.1e-06","5.0e-06","5.7e-06","5.8e-06","6.0e-06","6.9e-06","7.6e-06","8.8e-06","8.9e-06","1.1e-05","1.1e-05","1.2e-05","1.4e-05","1.6e-05","1.7e-05","1.7e-05","2.1e-05","2.3e-05","2.7e-05","2.9e-05","2.9e-05","3.2e-05","3.3e-05","3.5e-05","4.9e-05","5.7e-05","6.1e-05","6.1e-05","6.1e-05","6.2e-05","6.3e-05","6.4e-05","6.4e-05","6.5e-05","7.5e-05","7.5e-05","7.9e-05","8.3e-05","8.9e-05","9.0e-05","1.0e-04","1.0e-04","1.0e-04","1.1e-04","1.2e-04","1.3e-04","1.4e-04","1.6e-04","1.7e-04","1.8e-04","1.9e-04","2.0e-04","2.1e-04","2.2e-04","2.2e-04","2.3e-04","2.3e-04","2.3e-04","2.5e-04","2.7e-04","2.7e-04","3.2e-04","3.8e-04","4.2e-04","4.2e-04","4.7e-04","5.6e-04","5.7e-04","7.7e-04","8.2e-04","8.5e-04","8.6e-04","8.8e-04","1.0e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.9e-03","2.0e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.4e-03","2.7e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","2.9e-03","3.1e-03","3.1e-03","3.3e-03","3.5e-03","3.5e-03","3.8e-03","3.8e-03","4.1e-03","4.4e-03","4.6e-03","4.6e-03","5.1e-03","5.2e-03","5.3e-03","5.5e-03","5.8e-03","5.8e-03","5.9e-03","6.2e-03","6.2e-03","6.3e-03","7.7e-03","7.8e-03","7.9e-03","8.8e-03","8.9e-03","9.0e-03","9.1e-03","9.2e-03","9.4e-03","9.6e-03","9.8e-03","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.2e-02","2.2e-02","2.2e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.7e-02","2.7e-02","2.9e-02","2.9e-02","3.0e-02","3.0e-02","3.0e-02","3.1e-02","3.1e-02","3.5e-02","3.5e-02","3.5e-02","3.6e-02","3.6e-02","3.6e-02","3.7e-02","3.8e-02","3.9e-02","4.0e-02","4.0e-02","4.1e-02","4.2e-02","4.3e-02","4.3e-02","4.3e-02","4.5e-02","4.5e-02","4.5e-02","4.6e-02","4.7e-02","4.7e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.9e-02"],[1951,5361,5541,5579,5579,5579,5102,4453,1011,5094,131,237,13117,11951,130,7607,181,14808,544,202,129,333,694,1236,4252,58,58,34,34,112,112,12046,159,1723,85,37,38,183,183,186,151,1304,57,222,119,6210,64,79,50,838,66,66,14027,51,6291,353,356,1325,28,4112,4136,289,997,1005,6880,427,229,246,701,3519,766,386,420,14767,3377,431,1470,2339,3148,152,6661,2670,1473,1866,523,116,117,388,1754,2216,880,171,125,2048,2779,149,13201,2107,3104,135,137,138,138,138,995,99,74,1842,496,141,203,1856,2778,15,928,1377,78,471,804,132,58,81,974,154,117,85,260,119,1867,1388,120,123,834,160,180,106,174,92,153,573,2142,2657,10,127,36,2289,154,727,2,2337,5,31,975,14,123,55,617,2234,55,15,1264,623,219,43,1577,6576,201,998,51,51,956,636,110,294,18,9829,191,1098,120,54,121,21,1081,123,167,247,25,219,664,1001,24,4651,44,661,49,955,3262,2178,77,214,1500,4058,6,863,6,26,628,6938,2,1384,3498,1284,326,45,5929,6192,5752,2,1010,84,473,148,58,433,11,12,698,2263,2265,2265,96,1564,2381,7,7,16,16,29,2017,11,39,348,444,171,5218,317,3,2516,2176,60,2177,435,6,102,12,178,77,3,106,534,215,194,620,144,5861,177,468,3,3,408,290,97,97,74,49,111,3,44,5,8,553,43,1932,4,573,58,12,416,103,296,426,8,5364,27,365,115],[65,108,110,104,104,104,97,91,37,96,18,20,165,129,17,120,17,173,24,19,16,19,30,40,83,10,10,10,10,14,14,154,15,47,11,10,10,17,17,17,15,38,10,18,13,102,10,11,9,26,10,10,165,9,102,20,20,27,7,76,79,19,23,23,108,18,15,17,23,67,19,17,17,166,64,21,40,36,59,12,102,53,32,42,21,10,10,19,40,46,29,10,10,42,57,11,154,37,58,10,10,10,10,10,28,6,8,40,15,10,12,40,55,5,23,26,8,18,26,10,5,8,27,10,9,8,13,9,42,35,9,6,26,10,13,8,11,8,8,21,36,48,4,4,5,47,6,18,2,28,3,4,24,4,9,6,21,27,4,3,27,21,9,3,25,89,10,27,6,6,26,21,8,13,3,98,9,28,8,6,8,3,28,8,9,10,4,10,21,18,5,58,4,9,5,23,53,36,6,11,24,21,3,18,3,5,16,88,2,19,55,30,7,4,87,83,85,2,26,7,15,8,5,13,2,3,18,36,36,36,7,24,45,3,3,3,3,3,38,3,4,10,13,8,72,11,2,33,30,5,30,7,2,7,3,6,6,2,3,14,9,9,8,8,78,8,13,2,2,11,12,6,6,6,5,7,2,4,2,3,11,2,27,2,13,4,3,12,7,10,10,2,71,2,12,7]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/BP-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/MF-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="ipev-1" class="section level3 tabset tabset-fade">
<h3><strong>5.2. IPEV</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in IPEV were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-a2454929a56e939b13f9" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-a2454929a56e939b13f9">{"x":{"data":[{"x":[126.401856989695,123.04211087035,78.8152089926941,79.4161391929022,120.302805671998,117.965421321838,122.991382996306,78.7839918394365,86.213674314737,79.1390869577413,79.4083349045878,78.7371661095502,84.8596302921901,131.872663098083,131.806326647411,75.7286129643523,75.7754386942387,75.7442215409811,75.7208086760379,78.7800896952793,134.86950981081,102.501224026872,79.4044327604306,137.936595118365,123.502563880899,175.030377476667,79.1703041109989,75.7403193968239,184.742814283927],"y":[3.04816361783985,2.95008516236631,2.91267864951687,2.82459124020021,2.81013580023506,2.76003564474058,2.56026804030291,2.46804307378359,2.4610331737463,2.44659894032836,2.42930815687234,2.42499811381757,2.4133125840382,2.33481279271341,2.08936084806694,1.89553897062704,1.89553897062704,1.89144965091826,1.87194094741366,1.77600359688685,1.68755142552807,1.6639703749094,1.65264889654685,1.6050941397307,1.58810662081418,1.50343623677018,1.49440338827728,1.48114233058905,1.31969564721022],"text":["GO:0045055 (790) <br> regulated exocytosis <br> 8.950e-04","GO:0043312 (483) <br> neutrophil degranulation <br> 1.122e-03","GO:0002283 (488) <br> neutrophil activation involved in immune response <br> 1.223e-03","GO:0002446 (500) <br> neutrophil mediated immunity <br> 1.498e-03","GO:0042119 (502) <br> neutrophil activation <br> 1.548e-03","GO:0036230 (509) <br> granulocyte activation <br> 1.738e-03","GO:0043299 (538) <br> leukocyte degranulation <br> 2.753e-03","GO:0002275 (552) <br> myeloid cell activation involved in immune response <br> 3.404e-03","GO:0006887 (906) <br> exocytosis <br> 3.459e-03","GO:0002366 (723) <br> leukocyte activation involved in immune response <br> 3.576e-03","GO:0002444 (558) <br> myeloid leukocyte mediated immunity <br> 3.721e-03","GO:0002263 (727) <br> cell activation involved in immune response <br> 3.758e-03","GO:0006508 (1851) <br> proteolysis <br> 3.861e-03","GO:0048024 (107) <br> regulation of mRNA splicing, via spliceosome <br> 4.626e-03","GO:0048002 (193) <br> antigen processing and presentation of peptide antigen <br> 8.140e-03","GO:0000377 (360) <br> RNA splicing, via transesterification reactions with bulged adenosine as nucleophile <br> 1.272e-02","GO:0000398 (360) <br> mRNA splicing, via spliceosome <br> 1.272e-02","GO:0000381 (60) <br> regulation of alternative mRNA splicing, via spliceosome <br> 1.284e-02","GO:0000375 (363) <br> RNA splicing, via transesterification reactions <br> 1.343e-02","GO:0002274 (671) <br> myeloid leukocyte activation <br> 1.675e-02","GO:0050684 (145) <br> regulation of mRNA processing <br> 2.053e-02","GO:0019882 (236) <br> antigen processing and presentation <br> 2.168e-02","GO:0002443 (888) <br> leukocyte mediated immunity <br> 2.225e-02","GO:0051702 (99) <br> biological process involved in interaction with symbiont <br> 2.483e-02","GO:0043484 (152) <br> regulation of RNA splicing <br> 2.582e-02","GO:1902415 (9) <br> regulation of mRNA binding <br> 3.137e-02","GO:0002376 (3438) <br> immune system process <br> 3.203e-02","GO:0000380 (76) <br> alternative mRNA splicing, via spliceosome <br> 3.303e-02","GO:1905214 (11) <br> regulation of RNA binding <br> 4.790e-02"],"key":["GO:0045055","GO:0043312","GO:0002283","GO:0002446","GO:0042119","GO:0036230","GO:0043299","GO:0002275","GO:0006887","GO:0002366","GO:0002444","GO:0002263","GO:0006508","GO:0048024","GO:0048002","GO:0000377","GO:0000398","GO:0000381","GO:0000375","GO:0002274","GO:0050684","GO:0019882","GO:0002443","GO:0051702","GO:0043484","GO:1902415","GO:0002376","GO:0000380","GO:1905214"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.3072888314456,18.654666539476,18.6686201748074,18.7014823158046,18.7068756345731,18.7255693982465,18.8001372494773,18.8345831552502,19.484190719191,19.1917768671192,18.8490554993779,19.1989933043445,20.3761491613802,16.4479601286341,17.3543795238725,18.2507785031483,18.2507785031483,15.491084538085,18.2623370544314,19.0938152500805,16.9227691640758,17.649930300511,19.4584043531925,16.3236657355402,16.9949009807963,11.5676686384874,21.1120679823212,15.8912347257268,12.0709458293364],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData659048e6","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[64.5244030636107,59.4896732535677,69.6488993741428,62.1358335723345,64.4307336717959,68.6497591947854,65.1176425451041,62.120222007032,62.2880463340335,54.9037759459705,62.1475422463113,59.4623530142884,64.4346365631215,62.1241248983576,69.7191514180038,58.0338947891135,60.8478794348816,63.9662896040478,59.4935761448933,71.15541542583,69.7230543093295,64.7624794344732,68.5717013682731,61.7377386571218,63.7789508204183],"y":[3.82590959487933,3.61061085503493,3.61025448401289,3.60529617252658,3.60529617252658,3.41456218361424,2.98263991178681,2.61176049639365,2.56069004541781,2.44098286935725,2.41257913524061,2.41257913524061,2.41257913524061,2.36833183904822,2.09555284005382,2.0538361836432,2.04310228682187,2.01300423195845,1.99810715938373,1.96175831253332,1.59511774623012,1.48480760419815,1.448330669768,1.33695279796815,1.33695279796815],"text":["GO:0070062 (2178) <br> extracellular exosome <br> 1.493e-04","GO:0031982 (4058) <br> vesicle <br> 2.451e-04","GO:1903561 (2263) <br> extracellular vesicle <br> 2.453e-04","GO:0043230 (2265) <br> extracellular organelle <br> 2.481e-04","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 2.481e-04","GO:0101002 (184) <br> ficolin-1-rich granule <br> 3.850e-04","GO:0070820 (163) <br> tertiary granule <br> 1.041e-03","GO:0043226 (14027) <br> organelle <br> 2.445e-03","GO:0043564 (3) <br> Ku70:Ku80 complex <br> 2.750e-03","GO:0005615 (3595) <br> extracellular space <br> 3.623e-03","GO:0043233 (5579) <br> organelle lumen <br> 3.867e-03","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 3.867e-03","GO:0070013 (5579) <br> intracellular organelle lumen <br> 3.867e-03","GO:0043227 (13201) <br> membrane-bounded organelle <br> 4.282e-03","GO:1904724 (55) <br> tertiary granule lumen <br> 8.025e-03","GO:0030141 (857) <br> secretory granule <br> 8.834e-03","GO:0034774 (320) <br> secretory granule lumen <br> 9.055e-03","GO:0060205 (324) <br> cytoplasmic vesicle lumen <br> 9.705e-03","GO:0031983 (326) <br> vesicle lumen <br> 1.004e-02","GO:1990904 (694) <br> ribonucleoprotein complex <br> 1.092e-02","GO:1904813 (124) <br> ficolin-1-rich granule lumen <br> 2.540e-02","GO:0070419 (9) <br> nonhomologous end joining complex <br> 3.275e-02","GO:0099503 (1027) <br> secretory vesicle <br> 3.562e-02","GO:0042470 (109) <br> melanosome <br> 4.603e-02","GO:0048770 (109) <br> pigment granule <br> 4.603e-02"],"key":["GO:0070062","GO:0031982","GO:1903561","GO:0043230","GO:0065010","GO:0101002","GO:0070820","GO:0043226","GO:0043564","GO:0005615","GO:0043233","GO:0031974","GO:0070013","GO:0043227","GO:1904724","GO:0030141","GO:0034774","GO:0060205","GO:0031983","GO:1990904","GO:1904813","GO:0070419","GO:0099503","GO:0042470","GO:0048770"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.5726305449007,21.3038880613148,20.6185360204335,20.6195937895567,20.6195937895567,17.2832606049772,17.1010792375046,22.6771653543307,7.82319528083224,21.1639380777036,21.6663975662515,21.6663975662515,21.6663975662515,22.6122969037459,15.3403081032901,19.4126398523241,18.0857237964723,18.1032217578591,18.1118819820541,19.1381259978836,16.6805170897473,11.5676686384874,19.6443241495253,16.4774015552466,16.4774015552466],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData659048e6","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[3.99409256943449,29.8314446285847,26.7329915950603,16.6610680692082,2.13267934904261,48.6133771857045,37.8507405780717,28.0988084234401,3.70531986857705,9.98027261153333,47.7938870886766],"y":[2.89536304456159,2.2947128033779,1.84150614490575,1.7778067058385,1.70375258630614,1.70201012898952,1.5974176798846,1.53361426492663,1.50581698435299,1.36434256881288,1.34690058349501],"text":["GO:0003824 (5682) <br> catalytic activity <br> 1.272e-03","GO:0045027 (3) <br> DNA end binding <br> 5.073e-03","GO:0035613 (14) <br> RNA stem-loop binding <br> 1.440e-02","GO:0016787 (2357) <br> hydrolase activity <br> 1.668e-02","GO:0000166 (2176) <br> nucleotide binding <br> 1.978e-02","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 1.986e-02","GO:0051575 (6) <br> 5'-deoxyribose-5-phosphate lyase activity <br> 2.527e-02","GO:0042802 (2063) <br> identical protein binding <br> 2.927e-02","GO:0003723 (1951) <br> RNA binding <br> 3.120e-02","GO:0008233 (647) <br> peptidase activity <br> 4.322e-02","GO:0120225 (2) <br> coenzyme A binding <br> 4.499e-02"],"key":["GO:0003824","GO:0045027","GO:0035613","GO:0016787","GO:0000166","GO:1901265","GO:0051575","GO:0042802","GO:0003723","GO:0008233","GO:0120225"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[21.6870079142129,7.82319528083224,12.6380259214685,20.6671991075244,20.5715274259391,20.5720791211568,10.4356445073215,20.5073705235963,20.4399508009906,19.0457837593899,3.77952755905512],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData659048e6","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,49.9206590071537],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.8226750487752,71.2217645783655],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.123780619987,193.971385215695],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.873401257316,200.003902016042],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[25.9603295035768,62.5222198135703,134.547582917841,198.938651636679],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"44b811f7e06":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"44b815a26f16":{"x":{},"xend":{},"y":{},"yend":{}},"44b86804e1b":{"x":{},"xend":{},"y":{},"yend":{}},"44b86dfc3939":{"x":{},"xend":{},"y":{},"yend":{}},"44b844c746fd":{"x":{},"xend":{},"y":{},"yend":{}},"44b85bf724f6":{"x":{},"y":{}},"44b86bd243b9":{"yintercept":{}}},"cur_data":"44b811f7e06","visdat":{"44b811f7e06":["function (y) ","x"],"44b815a26f16":["function (y) ","x"],"44b86804e1b":["function (y) ","x"],"44b86dfc3939":["function (y) ","x"],"44b844c746fd":["function (y) ","x"],"44b85bf724f6":["function (y) ","x"],"44b86bd243b9":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData659048e6"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))
}</code></pre>
<div id="htmlwidget-2b7856457827ab1876f7" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2b7856457827ab1876f7">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:MF","GO:CC","GO:CC","GO:BP"],[2743,1453,4056,2131,2719,3800,13142,2895,12281,947,512,1101,11579,10980,2127,2170,12268,939,2843,1030,278,1099,927,2496,2134,1446,2720,2128,14544,7133,4074,14527,1080,1801,2600,1454,4442,156,168,160,154,6339,3758,938,35,11946,15312,7017,1098,16098,9188,4075,12399,6689,438,25604,1038,2804,159,3780,2046,11736,2029,2552,28093],["extracellular exosome","vesicle","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","ficolin-1-rich granule","regulated exocytosis","tertiary granule","neutrophil degranulation","neutrophil activation involved in immune response","catalytic activity","neutrophil mediated immunity","neutrophil activation","granulocyte activation","organelle","Ku70:Ku80 complex","leukocyte degranulation","myeloid cell activation involved in immune response","exocytosis","leukocyte activation involved in immune response","extracellular space","myeloid leukocyte mediated immunity","cell activation involved in immune response","proteolysis","organelle lumen","membrane-enclosed lumen","intracellular organelle lumen","membrane-bounded organelle","regulation of mRNA splicing, via spliceosome","DNA end binding","tertiary granule lumen","antigen processing and presentation of peptide antigen","secretory granule","secretory granule lumen","cytoplasmic vesicle lumen","vesicle lumen","ribonucleoprotein complex","RNA splicing, via transesterification reactions with bulged adenosine as nucleophile","mRNA splicing, via spliceosome","regulation of alternative mRNA splicing, via spliceosome","RNA splicing, via transesterification reactions","RNA stem-loop binding","hydrolase activity","myeloid leukocyte activation","nucleotide binding","nucleoside phosphate binding","regulation of mRNA processing","antigen processing and presentation","leukocyte mediated immunity","biological process involved in interaction with symbiont","5'-deoxyribose-5-phosphate lyase activity","ficolin-1-rich granule lumen","regulation of RNA splicing","identical protein binding","RNA binding","regulation of mRNA binding","immune system process","nonhomologous end joining complex","alternative mRNA splicing, via spliceosome","secretory vesicle","peptidase activity","coenzyme A binding","melanosome","pigment granule","regulation of RNA binding"],["1.5e-04","2.5e-04","2.5e-04","2.5e-04","2.5e-04","3.8e-04","9.0e-04","1.0e-03","1.1e-03","1.2e-03","1.3e-03","1.5e-03","1.5e-03","1.7e-03","2.4e-03","2.7e-03","2.8e-03","3.4e-03","3.5e-03","3.6e-03","3.6e-03","3.7e-03","3.8e-03","3.9e-03","3.9e-03","3.9e-03","3.9e-03","4.3e-03","4.6e-03","5.1e-03","8.0e-03","8.1e-03","8.8e-03","9.1e-03","9.7e-03","1.0e-02","1.1e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.7e-02","1.7e-02","2.0e-02","2.0e-02","2.1e-02","2.2e-02","2.2e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.9e-02","3.1e-02","3.1e-02","3.2e-02","3.3e-02","3.3e-02","3.6e-02","4.3e-02","4.5e-02","4.6e-02","4.6e-02","4.8e-02"],[2178,4058,2263,2265,2265,184,790,163,483,488,5682,500,502,509,14027,3,538,552,906,723,3595,558,727,1851,5579,5579,5579,13201,107,3,55,193,857,320,324,326,694,360,360,60,363,14,2357,671,2176,2177,145,236,888,99,6,124,152,2063,1951,9,3438,9,76,1027,647,2,109,109,11],[15,22,15,15,15,6,11,5,9,9,18,9,9,9,38,2,9,9,11,10,17,9,10,13,24,24,24,37,5,2,3,5,9,6,6,6,8,7,7,4,7,2,11,9,10,10,5,5,10,4,2,4,5,13,13,2,16,2,4,9,6,1,3,3,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/CC-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each protein-protein interaction is annotated with one or more ‘scores’. Importantly, these scores do <strong>not</strong> indicate the strength or the specificity of the interaction. Instead, they are indicators of <strong>confidence</strong>, i.e. how likely STRING judges an interaction to be true, given the available evidence. All scores rank from 0 to 1, with 1 being the highest possible confidence. A score of 0.5 would indicate that roughly every second interaction might be erroneous (i.e., a false positive).</p>
<pre class="r"><code>if(leng_dep != 0){
  
  string_db &lt;- STRINGdb$new(version=&quot;11&quot;, species=9606, score_threshold=400)

  string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)

  hits &lt;- string_mapped$STRING_id[1:leng_dep]

  string_db$plot_network(hits, add_link = T)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/stringdb-1.png" width="1440" style="display: block; margin: auto;" /></p>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex networks and integrating these with any type of attribute data. In this presentation all proteins showing at least one confidence level from the STRING database are shown. The <strong>color</strong> code is relative to <strong>fold change</strong> and the <strong>size</strong> is directly proportional to the <strong>significance (p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(&quot;assets/network_FC_pvalue.png&quot;,  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_FC_pvalue.png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network. Instead of nodes representing genes, nodes represent pathways or functions. Edges between these pathways or functions represent shared genes or pathway crosstalk. An enrichment map is a way to visualize your enrichment results to help reduce redundancy and uncover main themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(&quot;assets/network_enrichmentmap.png&quot;,  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_enrichmentmap.png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(&quot;output/results_table.xlsx&quot;), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.7                 purrr_0.3.4                
 [5] readr_1.4.0                 tidyr_1.1.3                
 [7] tibble_3.1.2                tidyverse_1.3.1            
 [9] openxlsx_4.2.4              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              naniar_0.6.1               
[15] visdat_0.5.3                ggrepel_0.9.1              
[17] ggplot2_3.3.5               UpSetR_1.4.0               
[19] ggstatsplot_0.8.0           datapasta_3.1.0            
[21] patchwork_1.1.1             DT_0.18                    
[23] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
[25] AnnotationDbi_1.52.0        gprofiler2_0.2.0           
[27] enrichR_3.0                 preprocessCore_1.52.1      
[29] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[31] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[33] IRanges_2.24.1              S4Vectors_0.28.1           
[35] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[37] matrixStats_0.59.0          DEP_1.12.0                 
[39] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3            R.methodsS3_1.8.1        
  [3] coda_0.19-4               bit64_4.0.5              
  [5] knitr_1.33                DelayedArray_0.16.2      
  [7] R.utils_2.10.1            data.table_1.14.0        
  [9] RCurl_1.98-1.3            doParallel_1.0.16        
 [11] generics_0.1.0            RSQLite_2.2.7            
 [13] correlation_0.6.1         chron_2.3-56             
 [15] bit_4.0.4                 lubridate_1.7.10         
 [17] xml2_1.3.2                httpuv_1.6.1             
 [19] assertthat_0.2.1          WRS2_1.1-2               
 [21] xfun_0.24                 hms_1.1.0                
 [23] jquerylib_0.1.4           evaluate_0.14            
 [25] promises_1.2.0.1          fansi_0.5.0              
 [27] readxl_1.3.1              caTools_1.18.2           
 [29] dbplyr_2.1.1              igraph_1.2.6             
 [31] DBI_1.1.1                 htmlwidgets_1.5.3        
 [33] reshape_0.8.8             kSamples_1.2-9           
 [35] Rmpfr_0.8-4               paletteer_1.3.0          
 [37] hash_2.2.6.1              ellipsis_0.3.2           
 [39] crosstalk_1.1.1           backports_1.2.1          
 [41] insight_0.14.2            gridBase_0.4-7           
 [43] vctrs_0.3.8               imputeLCMD_2.0           
 [45] Cairo_1.5-12.2            cachem_1.0.5             
 [47] withr_2.4.2               treemap_2.4-2            
 [49] cluster_2.1.0             lazyeval_0.2.2           
 [51] crayon_1.4.1              labeling_0.4.2           
 [53] pkgconfig_2.0.3           SuppDists_1.1-9.5        
 [55] slam_0.1-48               statsExpressions_1.1.0   
 [57] ProtGenerics_1.22.0       wordcloud_2.6            
 [59] rlang_0.4.11              RJSONIO_1.3-1.4          
 [61] lifecycle_1.0.0           MatrixModels_0.5-0       
 [63] sandwich_3.0-1            affyio_1.60.0            
 [65] BiocFileCache_1.14.0      modelr_0.1.8             
 [67] cellranger_1.1.0          rprojroot_2.0.2          
 [69] graph_1.68.0              Matrix_1.2-18            
 [71] mc2d_0.1-21               zoo_1.8-9                
 [73] reprex_2.0.0              whisker_0.4              
 [75] GlobalOptions_0.1.2       pheatmap_1.0.12          
 [77] png_0.1-7                 viridisLite_0.4.0        
 [79] rjson_0.2.20              mzR_2.24.1               
 [81] PMCMRplus_1.9.0           parameters_0.14.0        
 [83] bitops_1.0-7              shinydashboard_0.7.1     
 [85] R.oo_1.24.0               KernSmooth_2.23-17       
 [87] blob_1.2.1                shape_1.4.6              
 [89] multcompView_0.1-8        tmvtnorm_1.4-10          
 [91] ggsignif_0.6.2            scales_1.1.1             
 [93] memoise_2.0.0             magrittr_2.0.1           
 [95] plyr_1.8.6                gplots_3.1.1             
 [97] zlibbioc_1.36.0           compiler_4.0.3           
 [99] plotrix_3.8-1             pcaMethods_1.82.0        
[101] clue_0.3-59               cli_3.0.0                
[103] affy_1.68.0               XVector_0.30.0           
[105] pbapply_1.4-3             MASS_7.3-53              
[107] tidyselect_1.1.1          vsn_3.58.0               
[109] stringi_1.6.2             highr_0.9                
[111] yaml_2.2.1                GOSemSim_2.16.1          
[113] norm_1.0-9.5              MALDIquant_1.19.3        
[115] grid_4.0.3                sass_0.4.0               
[117] tools_4.0.3               rstudioapi_0.13          
[119] circlize_0.4.13           foreach_1.5.1            
[121] git2r_0.28.0              gridExtra_2.3            
[123] ipmisc_6.0.2              pairwiseComparisons_3.1.6
[125] farver_2.1.0              mzID_1.28.0              
[127] digest_0.6.27             BiocManager_1.30.16      
[129] shiny_1.6.0               proto_1.0.0              
[131] BWStest_0.2.2             Rcpp_1.0.6               
[133] broom_0.7.8               BayesFactor_0.9.12-4.2   
[135] performance_0.7.2         later_1.2.0              
[137] ncdf4_1.17                httr_1.4.2               
[139] MSnbase_2.15.7            ComplexHeatmap_2.6.2     
[141] effectsize_0.4.5          colorspace_2.0-2         
[143] rvest_1.0.0               XML_3.99-0.6             
[145] fs_1.5.0                  rematch2_2.1.2           
[147] gmm_1.6-6                 plotly_4.9.4.1           
[149] xtable_1.8-4              gmp_0.6-2                
[151] jsonlite_1.7.2            zeallot_0.1.0            
[153] NLP_0.2-1                 R6_2.5.0                 
[155] tm_0.7-8                  gsubfn_0.7               
[157] pillar_1.6.1              htmltools_0.5.1.1        
[159] mime_0.11                 glue_1.4.2               
[161] fastmap_1.1.0             BiocParallel_1.24.1      
[163] codetools_0.2-16          mvtnorm_1.1-2            
[165] utf8_1.2.1                lattice_0.20-41          
[167] bslib_0.2.5.1             sqldf_0.4-11             
[169] curl_4.3.2                gtools_3.9.2             
[171] zip_2.2.0                 GO.db_3.12.1             
[173] limma_3.46.0              rmarkdown_2.9            
[175] munsell_0.5.0             GetoptLong_1.0.5         
[177] GenomeInfoDbData_1.2.4    iterators_1.0.13         
[179] impute_1.64.0             haven_2.4.1              
[181] gtable_0.3.0              bayestestR_0.10.0        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
